<img 
src="template0x.png" alt="PICT" >
                                                                                

       <span 
class="cmr-12">pgfid11029889425470153 pgfid43805568225484898 pgfid54040744125216201</span>
<span 
class="cmr-12">pgfid6994622916357077 pgfid93805568216371822 pgfid104040744116103125</span>
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"  
  "http://www.w3.org/TR/html4/loose.dtd">  
<html > 
<head><title></title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"> 
<meta name="generator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<meta name="originator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<!-- html --> 
<meta name="src" content="template.tex"> 
<meta name="date" content="2014-04-24 22:41:00"> 
<link rel="stylesheet" type="text/css" href="template.css"> 
</head><body 
>
                                                                                <img 
src="template1x.png" alt="PICT" >
<img 
src="template2x.png" alt="PICT" >
                                                                                
<!--l. 14--><p class="indent" >       <a 
 id="likesection.1"></a><a 
 id="x1-2"></a><object data="template-1.svg" width="1.0 " height="2.33333 " type="image/svg+xml"><p>SVG-Viewer needed.</p></object>
<span class="marginpar"><object data="template-2.svg" width="96.69295 " height="76.77286 " type="image/svg+xml"><p>SVG-Viewer needed.</p></object>
<object data="template-3.svg" width="1.0 " height="2.33333 " type="image/svg+xml"><p>SVG-Viewer needed.</p></object>
</span>
<!--l. 50--><p class="indent" >
                                                                                <img 
src="template3x.png" alt="PICT" >
<img 
src="template4x.png" alt="PICT" >
                                                                                
<div class="center" 
>
<!--l. 50--><p class="noindent" >
<!--l. 50--><p class="noindent" >UNIVERSITY OF CALIFORNIA, SAN DIEGO
<!--l. 50--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Metagenomics approaches in the study of hypersaline microbial</tspan>
<tspan font-family="cmbx" font-size="12">communities</tspan>
<!--l. 50--><p class="noindent" >A dissertation submitted in partial satisfaction of the
<!--l. 50--><p class="noindent" >requirements for the degree
<!--l. 50--><p class="noindent" >Doctor of Philosophy
<!--l. 50--><p class="noindent" >in
<!--l. 50--><p class="noindent" >Marine Biology
<!--l. 50--><p class="noindent" >by
<!--l. 50--><p class="noindent" >Juan A. Ugalde Casanova
</div>
<!--l. 50--><p class="noindent" >Committee in charge:
      Dr. Eric E. Allen, Chair<br 
class="newline" /> Dr. Farooq Azam<br 
class="newline" />Dr. Douglas H. Bartlett<br 
class="newline" />Dr. Philip Bourne<br 
class="newline" />Dr. David T. Pride<br 
class="newline" />Dr. Forest Rohwer<br 
class="newline" />
<div class="center" 
>
<!--l. 50--><p class="noindent" >
<!--l. 50--><p class="noindent" >2014</div>
                                                                                <img 
src="template5x.png" alt="PICT" >
<img 
src="template6x.png" alt="PICT" >
                                                                                
<!--l. 50--><p class="indent" >
                                                                                <img 
src="template7x.png" alt="PICT" >
<img 
src="template8x.png" alt="PICT" >
                                                                                
<div class="center" 
>
<!--l. 50--><p class="noindent" >
<!--l. 50--><p class="noindent" >Copyright
<!--l. 50--><p class="noindent" >Juan A. Ugalde Casanova, 2014
<!--l. 50--><p class="noindent" >All rights reserved.</div>
                                                                                <img 
src="template9x.png" alt="PICT" >
<img 
src="template10x.png" alt="PICT" >
                                                                                
<!--l. 50--><p class="indent" >       <a 
 id="likesection.2"></a><a 
 id="Q1-1-0"></a>
                                                                                <img 
src="template11x.png" alt="PICT" >
<img 
src="template12x.png" alt="PICT" >
                                                                                
<div class="center" 
>
<!--l. 50--><p class="noindent" >
<!--l. 50--><p class="noindent" >The dissertation of Juan A. Ugalde Casanova is approved,
and it is acceptable in quality and form for publication
on microfilm and electronically:                        <br />
 <table id="TBL-1" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-1-1g"><col 
id="TBL-1-1"><col 
id="TBL-1-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-1-1-"><td  style="white-space:nowrap; text-align:right;" id="TBL-1-1-1"  
class="td10"> </td><td  style="white-space:nowrap; text-align:right;" id="TBL-1-1-2"  
class="td01">      </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-2-"><td  style="white-space:nowrap; text-align:right;" id="TBL-1-2-1"  
class="td10">  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-1-2-2"  
class="td01">      </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-3-"><td  style="white-space:nowrap; text-align:right;" id="TBL-1-3-1"  
class="td10">  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-1-3-2"  
class="td01">      </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-4-"><td  style="white-space:nowrap; text-align:right;" id="TBL-1-4-1"  
class="td10">  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-1-4-2"  
class="td01">      </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-5-"><td  style="white-space:nowrap; text-align:right;" id="TBL-1-5-1"  
class="td10">  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-1-5-2"  
class="td01">      </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-6-"><td  style="white-space:nowrap; text-align:right;" id="TBL-1-6-1"  
class="td10">  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-1-6-2"  
class="td01">      </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-7-"><td  style="white-space:nowrap; text-align:right;" id="TBL-1-7-1"  
class="td10">  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-1-7-2"  
class="td01">Chair  </td></tr></table><br /><br />
University of California, San Diego<br />
2014</div>
                                                                                <img 
src="template13x.png" alt="PICT" >
<img 
src="template14x.png" alt="PICT" >
                                                                                
<!--l. 67--><p class="indent" >       <a 
 id="likesection.3"></a><a 
 id="Q1-1-0"></a>
                                                                                <img 
src="template15x.png" alt="PICT" >
<img 
src="template16x.png" alt="PICT" >
                                                                                
       <div  
class="centerline">                                    DEDICATION                                            </div>
<div class="center" 
>
<!--l. 67--><p class="noindent" >
<!--l. 68--><p class="noindent" ><tspan font-family="cmr" font-size="12">To two, the loneliest number since the number one.</tspan></div>
                                                                                <img 
src="template17x.png" alt="PICT" >
<img 
src="template18x.png" alt="PICT" >
                                                                                
<!--l. 95--><p class="indent" >       <a 
 id="likesection.4"></a><a 
 id="Q1-1-0"></a>
                                                                                <img 
src="template19x.png" alt="PICT" >
<img 
src="template20x.png" alt="PICT" >
                                                                                
       <div  
class="centerline">                                             EPIGRAPH                                             </div>
                                                                                <div class="flushright" 
>
<!--l. 95--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A beginning is the time for taking the most delicate care that the balances are</tspan>
<tspan font-family="cmti" font-size="12">correct.</tspan><br />
&#8212;Frank Herbert, <tspan font-family="cmti" font-size="12">Dune</tspan>.</div>
                                                                                <img 
src="template21x.png" alt="PICT" >
<img 
src="template22x.png" alt="PICT" >
                                                                                
<!--l. 112--><p class="indent" >
                                                                                <img 
src="template23x.png" alt="PICT" >
<img 
src="template24x.png" alt="PICT" >
                                                                                
<!--l. 112--><p class="indent" >       <a 
 id="likesection.5"></a><a 
 id="Q1-1-0"></a><div  
class="centerline">                                     TABLE OF CONTENTS                                     </div>
                                                                                <img 
src="template25x.png" alt="PICT" >
<img 
src="template26x.png" alt="PICT" >
                                                                                
       <div  
class="centerline">                                        LIST OF FIGURES                                        </div><a 
 id="likesection.6"></a><a 
 id="Q1-1-0"></a>
             <!--l. 2--><p class="noindent" >Figure&#x00A0;1.1: Diagram comparing two possible approaches for the analysis of
             metagenomic data from natural microbial communities. Figure
             from Bragg and Tyson, 2014 <span class="cite">[<a 
href="#XBragg:2014kv">9</a>]</span>.   .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .     6
<!--l. 2--><p class="indent" >       figure.caption.12
             <!--l. 3--><p class="noindent" >Figure&#x00A0;1.2:  Comparison  of  a  set  of  reads  (Illumin  XXbp,  total  of  XX
             reads) classified using Phylosift? with two different reference
             databases. The first case shows the reads classified in abscence of
             any reference genomes, the second one where reference genomes
             obtained  through  metagenomics  <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>]</span>  are  included  in  the
             database .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .     6
<!--l. 3--><p class="indent" >       figure.caption.13
             <!--l. 4--><p class="noindent" >Figure&#x00A0;1.3: Phylogenetic tree of <tspan font-family="cmti" font-size="12">Candidatus </tspan>Halobonum tyrrellensis G22
             and related microorganisms, based on 16S rRNA sequences.  .  .   11
<!--l. 4--><p class="indent" >       figure.caption.14
             <!--l. 5--><p class="noindent" >Figure&#x00A0;1.4: Phylogenetic tree of <tspan font-family="cmti" font-size="12">Candidatus </tspan>Halobonum tyrrellensis G22
             and  related  microorganisms,  based  on  phylogenetic  marker
             sequences implemented in the Phylophlan software <span class="cite">[<a 
href="#XSegata:2013hb">76</a>]</span>  .  .  .  .  .   12
<!--l. 5--><p class="indent" >       figure.caption.15
             <!--l. 6--><p class="noindent" >Figure&#x00A0;1.5: Ionic composition of several aquatic systems. Figure from Oren,
             2013. <span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   13
<!--l. 6--><p class="indent" >       figure.caption.16
             <!--l. 7--><p class="noindent" >Figure&#x00A0;1.6:  Salt  concentration  limits  for  some  microbial  metabolic
             processes. Black bars indicate information based on laboratory
             studies, while white bars indicate activities measured in natural
             microbial communities. Figure from Oren, 2013 <span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>.   .  .  .  .  .  .   13
<!--l. 7--><p class="indent" >       figure.caption.17
             <!--l. 8--><p class="noindent" >Figure&#x00A0;1.7: Location of Lake Tyrrell in the southeastern region of Australia.
             Figure from Macumber, 1992. <span class="cite">[<a 
href="#XMacumber:1992ty">45</a>]</span>.  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   17
<!--l. 8--><p class="indent" >       figure.caption.20
             <!--l. 13--><p class="noindent" >Figure&#x00A0;5.1: Classification of the reference SNPs, by type.  .  .  .  .  .  .  .  .  .  .   64
<!--l. 13--><p class="indent" >       figure.caption.22
             <!--l. 14--><p class="noindent" >Figure&#x00A0;5.2:           Comparison           between           the           total
             number and the non-synonymous SNPs found in each genome,
             associated to functional categories from the COG classification     65
<!--l. 14--><p class="indent" >       figure.caption.23 subfigure.2.1 subfigure.2.2
             <!--l. 17--><p class="noindent" >Figure&#x00A0;5.3: Count of non-synonymous SNPs found in COGs categories from
             the Cellular Processes and Signaling group .  .  .  .  .  .  .  .  .  .  .  .   66
<!--l. 17--><p class="indent" >       figure.caption.24
             <!--l. 18--><p class="noindent" >Figure&#x00A0;5.4: Count of non-synonymous SNPs found in COGs categories from
             the Information Storage and Processing group  .  .  .  .  .  .  .  .  .  .   66
<!--l. 18--><p class="indent" >       figure.caption.25
                                                                                <img 
src="template27x.png" alt="PICT" >
<img 
src="template28x.png" alt="PICT" >
                                                                                
             <!--l. 19--><p class="noindent" >Figure&#x00A0;5.5: Count of non-synonymous SNPs found in COGs categories from
             the Metabolism group  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   67
<!--l. 19--><p class="indent" >       figure.caption.26
             <!--l. 20--><p class="noindent" >Figure&#x00A0;5.6: Analysis pipeline workflow   .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   67
<!--l. 20--><p class="indent" >       figure.caption.27
             <!--l. 21--><p class="noindent" >Figure&#x00A0;5.7: Mapped reads per genome   .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   67
<!--l. 21--><p class="indent" >       figure.caption.28
             <!--l. 22--><p class="noindent" >Figure&#x00A0;5.8: Depth of coverage for each genome .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   67
<!--l. 22--><p class="indent" >       figure.caption.29
             <!--l. 23--><p class="noindent" >Figure&#x00A0;5.9: SNPs in COGs .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   67
<!--l. 23--><p class="indent" >       figure.caption.30
             <!--l. 24--><p class="noindent" >Figure&#x00A0;5.10: Transition transversion rates  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   67
<!--l. 24--><p class="indent" >       figure.caption.31
             <!--l. 25--><p class="noindent" >Figure&#x00A0;5.11: Network diagram SNPs  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   67
<!--l. 25--><p class="indent" >       figure.caption.32
             <!--l. 26--><p class="noindent" >Figure&#x00A0;5.12: Map of SNPs in contigs  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   68
<!--l. 26--><p class="indent" >       figure.caption.33
             <!--l. 27--><p class="noindent" >Figure&#x00A0;5.13: SNP rarefaction curve per population  .  .  .  .  .  .  .  .  .  .  .  .  .  .   68
<!--l. 27--><p class="indent" >       figure.caption.34
             <!--l. 28--><p class="noindent" >Figure&#x00A0;5.14: Calculation of pN/pS ratios .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   68
<!--l. 28--><p class="indent" >       figure.caption.35
             <!--l. 29--><p class="noindent" >Figure&#x00A0;5.15: Histogram of dN/dS ratios  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   68
<!--l. 29--><p class="indent" >       figure.caption.36
             <!--l. 31--><p class="noindent" >Figure&#x00A0;6.1: Overview of the pipeline developed for the comparative analysis
             of microbial genomes .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   71
<!--l. 31--><p class="indent" >       figure.caption.37
             <!--l. 32--><p class="noindent" >Figure&#x00A0;6.2: Preprocessing step .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   72
<!--l. 32--><p class="indent" >       figure.caption.38
             <!--l. 33--><p class="noindent" >Figure&#x00A0;6.3: HQPanGenome   .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   76
<!--l. 33--><p class="indent" >       figure.caption.43
             <!--l. 34--><p class="noindent" >Figure&#x00A0;6.4: Circular plot  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   77
<!--l. 34--><p class="indent" >       figure.caption.44
                                                                                <img 
src="template29x.png" alt="PICT" >
<img 
src="template30x.png" alt="PICT" >
                                                                                
       <div  
class="centerline">                                         LIST OF TABLES                                         </div><a 
 id="likesection.7"></a><a 
 id="Q1-1-0"></a>
            <!--l. 2--><p class="noindent" >Table&#x00A0;1.1: Halophilic Archaea, modified from <span class="cite">[<a 
href="#XVentosa:2012wo">89</a>]</span> to include the recently
            discovered <tspan font-family="cmti" font-size="12">Nanohaloarchaea </tspan><span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>. .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   10
<!--l. 2--><p class="indent" >       table.caption.18
            <!--l. 3--><p class="noindent" >Table&#x00A0;1.2: Halophilic Bacteria, modified from <span class="cite">[<a 
href="#XVentosa:2012wo">89</a>]</span>. .  .  .  .  .  .  .  .  .  .  .  .  .  .   14
<!--l. 3--><p class="indent" >       table.caption.19
            <!--l. 8--><p class="noindent" >Table&#x00A0;5.1: Genome and reference SNPs  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   63
<!--l. 8--><p class="indent" >       table.caption.21
            <!--l. 10--><p class="noindent" >Table&#x00A0;6.1: Genes under selection in HQ strains   .  .  .  .  .  .  .  .  .  .  .  .  .  .  .   77
<!--l. 10--><p class="indent" >       table.caption.45
<!--l. 126--><p class="indent" >
                                                                                <img 
src="template31x.png" alt="PICT" >
<img 
src="template32x.png" alt="PICT" >
                                                                                
<!--l. 126--><p class="indent" >       <a 
 id="likesection.8"></a><a 
 id="Q1-1-0"></a><div  
class="centerline">                                    ACKNOWLEDGEMENTS                                    </div>
                                                                                <img 
src="template33x.png" alt="PICT" >
<img 
src="template34x.png" alt="PICT" >
                                                                                
<!--l. 136--><p class="indent" >
                                                                                <img 
src="template35x.png" alt="PICT" >
<img 
src="template36x.png" alt="PICT" >
                                                                                
<!--l. 136--><p class="indent" >       <a 
 id="likesection.9"></a><a 
 id="Q1-1-0"></a>
<div class="center" 
>
<!--l. 137--><p class="noindent" >
<!--l. 137--><p class="noindent" >VITA</div>
2001-2003           Research assistant, Laboratory of Bioinformatics and Gene
                     Expression, Universidad de Chile, Santiago, Chile.
2003-2004           Research   assistant,   Whitney   Laboratory   for   Marine
                     Biosciences, University of Florida.
2006                <tspan font-family="cmti" font-size="12">Licenciatura                                                         </tspan>in
                     Molecular Biotechnology Engineering. Universidad de Chile,
                     Santiago, Chile.
2004-2008           Project
                     Engineer, Laboratory of Bioinformatics and Mathematics of
                     the Genome, Universidad de Chile, Santiago, Chile.
2008-2012           Fulbright fellow.
2014                Doctor of Philosophy, Scripps Institution of Oceanography,
                     University of California, San Diego
<div class="center" 
>
<!--l. 147--><p class="noindent" >
<!--l. 147--><p class="noindent" >PUBLICATIONS</div>
Martin LJ, Adams R, Bateman A, Bik HM, Haws J, Hird SM, Hughes D, Kembel
SW, Kinney K, Kolokotronis SO, Levy G, Mclain C, Meadow JF, Medina RF,
Mhuireach G, Moreau CS, Munshi-South J, Nichols, LM, Palmer C, Popova L,
Schal C, Siegel J, Taubel M, Trautwein M, Ugalde JA, Dunn RR. Evolution in the
Indoor Biome. <tspan font-family="cmti" font-size="12">Proc R Soc B</tspan>. <tspan font-family="cmti" font-size="12">Accepted</tspan>.
                                                                                <img 
src="template37x.png" alt="PICT" >
<img 
src="template38x.png" alt="PICT" >
                                                                                
Valenzuela C, Ugalde JA, Mora GC, Alvarez S, Contreras I, Santiviago CA. Draft
Genome Sequence of <tspan font-family="cmti" font-size="12">Salmonella enterica </tspan>Serovar <tspan font-family="cmti" font-size="12">Typhi</tspan>. <tspan font-family="cmti" font-size="12">Genome Announc </tspan>2(1):
e00104-14. 2014.
Malfatti  F,  Turk  V,  Tinta  T,  Mozeti&#x010D;  P,  Manganelli  M,  Samo  TJ,  Ugalde
JA,  Kova&#x010D;  N,  Stefanelli  M,  Antonioli  M,  Fonda-Umani  S,  Del  Negro  P,
Cataletto  B,  Hozi&#263;  A,  Ivo&#353;evi&#263;  DeNardis  N,  Mi&#353;i&#263;  Radi&#263;  T,  Radi&#263;  T,  Fuks
D,  Azam  F.  Microbial  mechanisms  coupling  carbon  and  phosphorus  cycles  in
phosphorous-limited  northern  Adriatic  Sea.  <tspan font-family="cmti" font-size="12">Sci Total Environ  </tspan>470:  1173-1183.
2014.
Ugalde JA, Narasingarao P, Kuo P, Podell S, Allen EE. Draft genome sequence of
&#8221;<tspan font-family="cmti" font-size="12">Candididatus </tspan>Halobonum tyrrellensis&#8221; Strain G22, Isolated from the Hypersaline
Waters of Lake Tyrrell, Australia. <tspan font-family="cmti" font-size="12">Genome Announc </tspan>1(6): e01001-13. 2013.
Podell S, Emerson JB, Jones, CM, Ugalde JA, Welch, S, Heidelberg KB, Banfield
JF,  Allen  EE.  Seasonal  fluctuations  in  ionic  concentrations  drive  microbial
succession in a hypersaline lake community. <tspan font-family="cmti" font-size="12">ISME J</tspan>, advance online publication.
doi:10.1038/ismej.2013.221.
Kharbush JJ, Ugalde JA, Hogle SL, Allen EE, Aluwihare LI. Composite bacterial
hopanoids  and  their  microbial  producers  across  oxygen  gradients  in  the  water
column of the California Current. <tspan font-family="cmti" font-size="12">Appl Env Microbiol </tspan>79(23): 7491-7501. 2013.
Ugalde JA, Gallard MJ, Belmar C, Muñoz P, Ruiz-Tagle N, Ferrada-Fuentes S,
Espinoza  C,  Allen  EE,  Gallardo  VA.  Microbial  Life  in  a  Fjord:  Metagenomic
Analysis of a Microbial Mat in Chilean Patagonia. <tspan font-family="cmti" font-size="12">PLoS One </tspan>8(8):e71952. 2013.
Podell S, Ugalde JA, Narasingarao P, Banfield JF, Heidelberg EE. Assembly-driven
community genomics of a hypersaline microbial ecosystem. <tspan font-family="cmti" font-size="12">PLoS One </tspan>8(4):e61692.
2013.
Narasingaro P, Podell S, Ugalde JA, Brochier-Armanet C, Emerson JB, Brocks
JJ, Heidelberg KB, Banfield JF, Allen EE. De novo metagenomic assembly reveals
abundant novel major lineage of Archaea in hypersaline microbial communities.
<tspan font-family="cmti" font-size="12">ISME J </tspan>6:81-93. 2012.
                                                                                <img 
src="template39x.png" alt="PICT" >
<img 
src="template40x.png" alt="PICT" >
                                                                                
Ugalde JA, Podell S, Narasingarao P, Allen EE. Xenorhodopsins, an enigmatic
new class of microbial rhodopsins horizontally transferred between Archaea and
Bacteria. <tspan font-family="cmti" font-size="12">Biol Direct </tspan>6:52. 2011.
Levicán G, Ugalde JA, Ehrenfeld N, Maass A, Parada P. Comparative genomic
analysis  of  carbon  and  nitrogen  assimilation  mechanisms  in  three  indigenous
bioleaching bacteria: predictions and validations. <tspan font-family="cmti" font-size="12">BMC Genomics </tspan>9(1):581. 2008.
Chang   BSW,   Ugalde   JA,   Matz   MV.   Applications   of   ancestral   protein
reconstruction  in  understanding  protein  function:  GFP-like  proteins.  <tspan font-family="cmti" font-size="12">Meth</tspan>
<tspan font-family="cmti" font-size="12">Enyzmol </tspan>395:652-670. 2005.
Matz  MV,  Labas  YA,  Ugalde  J.  Evolution  of  function  and  color  in  GFP-like
proteins. <tspan font-family="cmti" font-size="12">Method of Biochemical Analysis, Green Fluorescent Protein</tspan>. Chalfie M,
Kain SR, Eds. John Wiley &amp; Sons. 2005.
Ugalde  JA,  Chang  BSW,  Matz  MV.  Evolution  of  Coral  Pigments  Recreated.
<tspan font-family="cmti" font-size="12">Science </tspan>305(5689): 1433. 2004.
Shagin DA, Barsova EV, Yanushevich YG, Fradkov AF, Lukyanov KA, Labas YA,
Semenova TN, Ugalde JA, Meyers A, Nunez JM, Widder EA, Lukyanov SA, Matz
MV. GFP-like proteins as ubiquitous metazoan superfamily: evolution of functional
features and structural complexity. <tspan font-family="cmti" font-size="12">Mol Biol Evol </tspan>21(5): 841-850. 2004.
<!--l. 188--><p class="indent" >
                                                                                <img 
src="template41x.png" alt="PICT" >
<img 
src="template42x.png" alt="PICT" >
                                                                                
<!--l. 188--><p class="indent" >       <a 
 id="likesection.10"></a><a 
 id="Q1-1-0"></a>
<div class="center" 
>
<!--l. 188--><p class="noindent" >
<!--l. 188--><p class="noindent" >ABSTRACT OF THE DISSERTATION
<!--l. 188--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Metagenomics approaches in the study of hypersaline microbial</tspan>
<tspan font-family="cmbx" font-size="12">communities</tspan>
<!--l. 188--><p class="noindent" >by
<!--l. 188--><p class="noindent" >Juan A. Ugalde Casanova
<!--l. 188--><p class="noindent" >Doctor of Philosophy in Marine Biology
<!--l. 188--><p class="noindent" >University of California, San Diego, 2014
<!--l. 188--><p class="noindent" >Dr. Eric E. Allen, Chair </div>
<!--l. 189--><p class="indent" >       This dissertation will be abstract.
                                                                                <img 
src="template43x.png" alt="PICT" >
<img 
src="template44x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template45x.png" alt="PICT" >
<img 
src="template46x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template47x.png" alt="PICT" >
<img 
src="template48x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template49x.png" alt="PICT" >
<img 
src="template50x.png" alt="PICT" >
                                                                                
<a 
 id="x1-3r1"></a>
<a 
 id="Q1-1-0"></a>
<!--l. 2--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Chapter</tspan>
<tspan font-family="cmbx" font-size="12">1</tspan>
<!--l. 2--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Introduction</tspan>
<tspan font-family="cmbx" font-size="12">to</tspan>
<tspan font-family="cmbx" font-size="12">the</tspan>
<tspan font-family="cmbx" font-size="12">thesis</tspan>
<!--l. 4--><p class="indent" >       Bacteria and Archaea represent an abundant component of Earth&#8217;s biomass, with
estimates between 9.2-31.7 10<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">29</tspan></sup> cells <span class="cite">[<a 
href="#XKallmeyer:2012km">35</a>]</span> to 41.8-64.3 10<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">29</tspan></sup> cells <span class="cite">[<a 
href="#XWhitman:1998tj">92</a>]</span> globally. Species
diversity estimates suggest that there are close to 10<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">7</tspan></sup> different species of Bacteria and
Archaea<span class="cite">[<a 
href="#XCurtis:2002dj">17</a>]</span>, although some authors suggest this number is an over-estimation
<span class="cite">[<a 
href="#XSchloss:2004do">74</a>]</span>. Nevertheless, Bacteria and Archaea represent the most diverse group of
organisms on Earth, and yet we have scarcely begun to characterize this diversity
<span class="cite">[<a 
href="#XWu:2009ju">97</a>,&#x00A0;<a 
href="#XRinke:2013bt">70</a>]</span>.
<!--l. 6--><p class="indent" >       Our inability to cultivate most of the microorganisms present in the environment
limits our understanding of the phylogenetic and functional diversity of microbial
communities. Known as the &#8221;Great Plate-Count Anomaly&#8221; <span class="cite">[<a 
href="#XStaley:1985ww">81</a>]</span>, our current culture
collections may not be representative of what can be observed in natural samples by
                                                                                <img 
src="template51x.png" alt="PICT" >
<img 
src="template52x.png" alt="PICT" >
                                                                                
culture-independent methods <span class="cite">[<a 
href="#XAmann:1995tw">3</a>]</span>. Most organisms are currently uncultured, due
to our lack of information on their physiology and nutritional requirements
<span class="cite">[<a 
href="#XStewart:2012dd">82</a>]</span>. With new and unique cultivation methods, and information derived from
genomic surveys <span class="cite">[<a 
href="#XTyson:2005by">84</a>]</span>, we may be able to culture these organisms in the near
future.
<!--l. 8--><p class="indent" >       Over the last decade, the development of advanced technologies and
experimental methods has allowed the study of natural microbial communities via
culture-independent techniques. Specifically, DNA-based methods, driven by the
development of next-generation sequencing <span class="cite">[<a 
href="#XMardis:2008fr">46</a>]</span>, have allowed the investigation of
natural microbial communities without relying on cultivation. DNA-based surveys span
two broad categories: marker-based analysis (such as the 16S rRNA gene) and whole
genome sequencing.
<!--l. 10--><p class="indent" >       Marker-based analysis provides a characterization,or census, of the
phylogenetic diversity present in the community, allowing for estimation of
community diversity<span class="cite">[<a 
href="#XCaporaso:2011cs">14</a>,&#x00A0;<a 
href="#XRappe:2003fc">68</a>,&#x00A0;<a 
href="#XCaporaso:2010bi">13</a>]</span> and identification of community members
based on sequence similarity against a reference database <span class="cite">[<a 
href="#XMcDonald:2011hn">49</a>]</span>. This approach
provides only a partial picture of the metabolic repertoire present in natural
microbial communities, as it requires extrapolation of genomic data from isolated
microorganisms to the entire community. Even strains with very similar 16S rRNA
sequences may have different metabolic and genetic properties, which are not
captured in the diversity found using a marker gene such as the 16S rRNA gene
<span class="cite">[<a 
href="#XJaspers:2004ge">32</a>]</span>.
<!--l. 12--><p class="indent" >       Metagenomics, also referred to as community genomics, relies on the direct
sequencing of genetic material isolated from a microbial community <tspan font-family="cmti" font-size="12">en masse </tspan><span class="cite">[<a 
href="#XWooley:2010uv">96</a>]</span>, and
provides the means to evaluate the phylogenetic and functional diversity of microbial
populations directly from the environment. Depending on the complexity of the
community under study, not all of the members of the microbial community will be
observed in the sequence data, as the most abundant members will dominate the
metagenomic analyses <span class="cite">[<a 
href="#XBragg:2014kv">9</a>]</span>. Using metagenomic methods, several authors report
studies of microbial communities with low to moderate species diversity (e.g.
<span class="cite">[<a 
href="#XUgalde:2013he">85</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XDinsdale:2008cd">20</a>,&#x00A0;<a 
href="#XTyson:2004bw">83</a>,&#x00A0;<a 
href="#XBrulc:2009fi">12</a>]</span>). Highly diverse communities, such as those found in soils and
                                                                                <img 
src="template53x.png" alt="PICT" >
<img 
src="template54x.png" alt="PICT" >
                                                                                
sediments, require high amounts of sequence information to provide a full picture of the
microorganisms that are present <span class="cite">[<a 
href="#XKakirde:2010ew">34</a>]</span>. The decrease in sequencing cost, driven by the
development of novel sequencing technologies, has enhanced our ability to deeply
sample complex communities using metagenomics approaches. Consequently,
the challenge will be developing or acquiring the computational resources and
algorithms necessary to analyze vast amounts of genetic sequence information
<span class="cite">[<a 
href="#XPell:2012id">63</a>,&#x00A0;<a 
href="#XKakirde:2010ew">34</a>]</span>.
<!--l. 14--><p class="indent" >       Microbial communities in extreme environments, such as those associated with
hypersaline habitats, represent tractable systems that can be studied using metagenomic
approaches. Because of their relatively low species diversity <span class="cite">[<a 
href="#XAndrei:2012he">5</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XGhai:2011hn">28</a>]</span>, it is
possible to study the phylogenetic and metabolic diversity that is present in
the community. In this first chapter, I provide an overview of metagenomics,
with particular emphasis in assembly-based approximations, followed by an
introduction into the microbiology of hypersaline environments and finally an overview
of our study site, the hypersaline waters of Lake Tyrrell located in Victoria,
Australia.
<a 
 id="x1-4r1"></a>
       <tspan font-family="cmbx" font-size="12">1.1</tspan>    <tspan font-family="cmbx" font-size="12">Metagenomics</tspan>
<a 
 id="Q1-1-0"></a>
<!--l. 18--><p class="noindent" >The study of natural microbial communities by analyzing DNA obtained directly from
environmental samples was first proposed by Handelsman <tspan font-family="cmti" font-size="12">et al. </tspan>in 1998 <span class="cite">[<a 
href="#XHandelsman:1998tx">29</a>]</span> to access
the genomic information of uncultured environmental microorganisms. The first
metagenomic studies were performed by isolating DNA from environmental samples,
plasmid-based cloning of this DNA and subsequence sequencing of these clones using the
Sanger method <span class="cite">[<a 
href="#XTyson:2004bw">83</a>,&#x00A0;<a 
href="#XVenter:2004hg">88</a>]</span>. Limited throughput, cloning biases, and the expesive
and laborious nature of this process limited the scope of early metagenomic
studies, often focused on microbial communities with low to medium species
diversity <span class="cite">[<a 
href="#XTyson:2004bw">83</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>]</span>, or to studies aimed at describing the general functional and
phylogenetic composition of a community <span class="cite">[<a 
href="#XVenter:2004hg">88</a>,&#x00A0;<a 
href="#XRusch:2007ez">73</a>]</span>. The development of <tspan font-family="cmti" font-size="12">next</tspan>
<tspan font-family="cmti" font-size="12">generation </tspan>sequencing technologies <span class="cite">[<a 
href="#XMardis:2013gn">47</a>]</span>, removed some of these limitations, allowing
for direct sequencing of DNA samples without cloning, higher throughputs,
                                                                                <img 
src="template55x.png" alt="PICT" >
<img 
src="template56x.png" alt="PICT" >
                                                                                
and a lower cost per base <span class="cite">[<a 
href="#XMardis:2013gn">47</a>]</span>. One of the remaining limitations in current
technologies is the length of the sequence reads, which are not yet close to the length
of reads generated by Sanger sequencing. Improvements to current methods
and the development of further advanced technologies, such as single molecule
sequencing <span class="cite">[<a 
href="#XEid:2009kv">22</a>]</span> and nanopores <span class="cite">[<a 
href="#XBranton:2008fr">10</a>]</span>, holds promise to further alleviate these
limitations.
<!--l. 20--><p class="indent" >       Metagenomic studies are driven, in most cases, by discovery, data mining and
comparative research, rather than by a specific hypothesis <span class="cite">[<a 
href="#XWooley:2010uv">96</a>]</span>. Accordingly, both the
sequence information and the contextual environmental data, or metadata, associated
with a sample are of great importance, allowing for both biological and physicochemical
context to be applied to the microbial community under study. These data include
environmental parameters associated with a sample and procedures used to process a
sample (e.g. filtration procedure, sample processing and library construction). All of
these variables become highly important in the bioinformatic analysis of a sample data
set<span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>.
<!--l. 22--><p class="indent" >       Based on the complexity of the microbial community under study, and the choice
of sampling and sequencing methods, two main approaches are used to study microbial
communities using metagenomics: gene-centric and assembly-based approaches (Figure
1.1). Gene-centric studies focus on complex communities or projects with a shallow level
of sequencing, where assembly of larger contiguous genome fragments from
community members is not feasible given the number of reads or size of the data
set obtained. . In these cases, the focus is on the phylogenetic and functional
diversity as a profile of the community <span class="cite">[<a 
href="#XUgalde:2013he">85</a>,&#x00A0;<a 
href="#XBrulc:2009fi">12</a>]</span>, and the comparison of different
environments based on these profiles <span class="cite">[<a 
href="#XDinsdale:2008cd">20</a>,&#x00A0;<a 
href="#XWillner:2009iv">95</a>,&#x00A0;<a 
href="#XReed:2014jj">69</a>]</span>. Gene-centric studies focus
on the phylogenetic and functional classification of the metagenomic reads,
either by analyzing every read present in the dataset <span class="cite">[<a 
href="#XBrady:2011hi">8</a>,&#x00A0;<a 
href="#XParks:2011bh">61</a>,&#x00A0;<a 
href="#XAmritaPati:2011ba">4</a>]</span>, or by using
marker-genes as a proxy to create either taxonomic and/or functional profiles of the
community <span class="cite">[<a 
href="#XDarling:2014ej">18</a>,&#x00A0;<a 
href="#XSegata:2012ba">77</a>]</span>. One main drawbacks of this method is that community
profiles are limited by the reference databases being used, which can lead to
missing unique groups that are present in the microbial community under
study.
                                                                                <img 
src="template57x.png" alt="PICT" >
<img 
src="template58x.png" alt="PICT" >
                                                                                
<!--l. 28--><p class="indent" >       Assembly-based metagenomics, involves the bioinformatic assembly of the
community sequence information, with the goal of recovering larger contiguous genome
fragments that can improve the phylogenetic and functional classification of
the organisms in the community. This approach allows the recovery of longer
fragments, generate better gene models and can identify previously unknown
microbial groups <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XKantor:2013ir">36</a>,&#x00A0;<a 
href="#XDiRienzi:2013kx">19</a>]</span>. The information obtained from the assembled
population genomes can be used to potentially guide cultivation efforts for these
organisms <span class="cite">[<a 
href="#XTyson:2005by">84</a>]</span>, in population genetic studies <span class="cite">[<a 
href="#XSimmons:2008by">80</a>,&#x00A0;<a 
href="#XWard:2008hp">91</a>,&#x00A0;<a 
href="#XAllen:2007ju">2</a>]</span>, and provide a better
picture of the interactions among the members of the microbial community
<span class="cite">[<a 
href="#XTyson:2004bw">83</a>,&#x00A0;<a 
href="#XFlowers:2013hq">26</a>]</span>. In addition, the recovery of near-complete genomes from metagenomics
datasets allows for the discovery of novel functions that may be missed by
viewing the unassembled sequence reads. For example, Chapter 2 of this thesis
shows how the assembly of sequence reads from the Lake Tyrrell metagenome
led to the discovery of a previously unknown Phylum of Archaea and to the
discovery of a novel family of microbial rhodopsins, called the xenorhodopsins, and
evidence that suggests its acquisition through horizontal gene transfer (Chapter
3).
<!--l. 30--><p class="indent" >       Assembly-based methods are limited by the complexity of the microbial
community under study, and the amount of sequencing needed to recover the genomes of
the most abundant members of the community. Furthermore, some of the more rare
members of the community are inaccessible by sequencing, because the most abundant
organisms dominate the metagenomic dataset. For example, estimations of the
number of sequences needed to address highly diverse samples, such as from
soils, show that billions of sequences are needed to be able to sample some
of these most abundant organisms <span class="cite">[<a 
href="#XHowe:2014go">30</a>]</span>. Even in simpler systems, the main
challenge is to classify the assembled genomic fragments into unique phylogenetic
bins, each representing a different population. By combining various pieces of
information, such as sample characteristics, nucleotide composition, amino
acid counts, and library abundance, it is possible to classify these genomic
fragments into unique populations, a process known as phylogenetic binning <span class="cite">[<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XAlbertsen:2013gp">1</a>]</span>.
Larger datasets represent a computational challenge because of the high memory
                                                                                <img 
src="template59x.png" alt="PICT" >
<img 
src="template60x.png" alt="PICT" >
                                                                                
requirements of short-read assembly software programs. In this case, the use of
methods for digital normalization may reduce the computational problems
<span class="cite">[<a 
href="#XPell:2012id">63</a>,&#x00A0;<a 
href="#XKakirde:2010ew">34</a>]</span>.
<!--l. 32--><p class="indent" >       The work presented in this thesis describes a study of the microbial community
inhabiting the hypersaline waters of Lake Tyrrell, Australia using an assembly-based
approach. The combination of a relatively low-species diversity, driven by the extreme
conditions found in this habitat <span class="cite">[<a 
href="#XAndrei:2012he">5</a>]</span>, and a deep sequencing approach, allowed for the
genomic reconstruction of some of the most abundant members of the community and
the discovery of novel microorganisms.
<!--l. 47--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template61x.png" alt="PICT" >
<img 
src="template62x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template63x.png" alt="PICT" >
<img 
src="template65x.png" alt="PICT" >
                                                                                
<!--l. 49--><p class="noindent" ><img 
src="template64x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template64x.png" src="Chapter1/Figures/Metagenomics_GeneCentric_vs_Assembly.pdf"  
--> <a 
 id="x1-5r1"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.1</tspan>:  Diagram  comparing  two  possible  approaches  for  the  analysis  of
metagenomic data from natural microbial communities. Figure from Bragg and
Tyson, 2014 <span class="cite">[<a 
href="#XBragg:2014kv">9</a>]</span>.
                                                                                <img 
src="template66x.png" alt="PICT" >
<img 
src="template67x.png" alt="PICT" >
                                                                                
<!--l. 52--><p class="indent" >       </div><hr class="endfigure">
<a 
 id="x1-6r2"></a>
<!--l. 61--><p class="noindent" ><tspan font-family="cmbx" font-size="12">1.2</tspan>    <tspan font-family="cmbx" font-size="12">Microbial communities in hypersaline</tspan>
        <tspan font-family="cmbx" font-size="12">environments</tspan>
<a 
 id="Q1-1-1"></a>
<!--l. 63--><p class="indent" >       Hypersaline habitats are found worldwide, in a variety of natural and man-made
environments. Examples include salt lakes, saline soils, salt ats, solar salterns, brine
pools, salted foods, and fermented foods <span class="cite">[<a 
href="#XOren:2002vw">55</a>]</span>. Aquatic hypersaline systems are the most
studied, and are either of marine origin (thalassohaline), or formed by the dissolution of
mineral salt deposits (athalassohaline).
<!--l. 65--><p class="indent" >       Within these saline systems, a variety of microbial species are adapted to these
environments. Moderate halophiles can beare found between 30-150 g/L NaCl, while
extreme halophiles exist in the range of 150-300 g/L NaCl <span class="cite">[<a 
href="#XAndrei:2012he">5</a>]</span>. In addition to the
high NaCl concentrations, other salts are also important to consider when
measuring the ionic composition of these systems, including Mg<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">2+</tspan></sup> and Ca<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">2+</tspan></sup>,
which can inuence the microbial community in these habitats <span class="cite">[<a 
href="#XMcGenity:2000ts">50</a>,&#x00A0;<a 
href="#XPodell:2013fp">66</a>]</span> (Figure
1.5).
<!--l. 67--><p class="indent" >       Genera from both the Bacteria and the Archaea exist in moderate and extreme
saline systems. Within the Archaea, the phylogenetic diversity appears to be limited to
the <tspan font-family="cmti" font-size="12">Euryarchaeota</tspan>, specifically in the classes <tspan font-family="cmti" font-size="12">Halobacteria </tspan>and <tspan font-family="cmti" font-size="12">Methanomicrobia </tspan>(Table
1.1) <span class="cite">[<a 
href="#XVentosa:2012wo">89</a>]</span>. The discovery of the <tspan font-family="cmti" font-size="12">Nanohaloarchaea </tspan>(described in Chapter 2, and <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>), a
novel Class, within the Euryarchaeota,, in globally dispersed hypersaline systems
expanded our understanding of the phylogenetic diversity of halophilic Archaea <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>.
Recently, a phylogenomic analysis of novel archaeal groups, isolated via single-cell
genomics, suggested that the <tspan font-family="cmti" font-size="12">Nanohaloarchaea </tspan>are a new phylum, sister to the
<tspan font-family="cmti" font-size="12">Euryarchaeota </tspan><span class="cite">[<a 
href="#XRinke:2013bt">70</a>]</span>, although more work (and more genomes and isolates) is
needed to fully resolve the phylogenetic relationships between these groups
<span class="cite">[<a 
href="#XWilliams:um">93</a>]</span>.
<!--l. 69--><p class="indent" >       Even within the <tspan font-family="cmti" font-size="12">Halobacteria</tspan>, novel taxonomic groups remain unidentified. Our
                                                                                <img 
src="template68x.png" alt="PICT" >
<img 
src="template69x.png" alt="PICT" >
                                                                                
group recently described the genome of a newly isolated halophilic Archaea, <tspan font-family="cmti" font-size="12">Candidatus</tspan>
Halobonum tyrrellensis strain G22 <span class="cite">[<a 
href="#XUgalde:2013hb">86</a>]</span>, which phylogenetic analysis suggests is a new
genus within the <tspan font-family="cmti" font-size="12">Halobacteria </tspan>(Appendix A). Analysis of the 16S rRNA gene, and a
phylogenomic approach using the markers genes implemented in the software,
PhyloPhlan, has further supported this phylogenetic placement (Figures 1.3 &amp;
1.4).
<!--l. 71--><p class="indent" >       The phylogenetic breadth of bacterial species found in saline systems is wider
than that of Archaea (Table 1.2). In moderate saline environments, Bacteria are more
abundant than Archaea <span class="cite">[<a 
href="#XOren:2008ej">56</a>,&#x00A0;<a 
href="#XGhai:2011hn">28</a>,&#x00A0;<a 
href="#XGhai:2012fb">27</a>,&#x00A0;<a 
href="#XCasamayor:2002tf">15</a>]</span>, but as salinity increases, Archaeal groups
become more abundant <span class="cite">[<a 
href="#XGhai:2011hn">28</a>,&#x00A0;<a 
href="#XCasamayor:2002tf">15</a>,&#x00A0;<a 
href="#XMutlu:2008dk">52</a>]</span>. One of the most abundant bacterial species found
in extreme hypersaline systems is <tspan font-family="cmti" font-size="12">Salinibacter ruber </tspan><span class="cite">[<a 
href="#XAnton:2008eb">6</a>]</span>. This bacterium shares many
phenotypic characteristics with halophilic Archaea <span class="cite">[<a 
href="#XOren:2013gy">60</a>]</span>, and multiple gene clusters
appear to have been acquired via horizontal gene transfer from the Archaea
<span class="cite">[<a 
href="#XMongodin:2005ie">51</a>]</span>.
<!--l. 73--><p class="indent" >       Viruses are also very abundant in hypersaline systems, with reports of counts of
at least 10<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">7</tspan></sup> viral-like particles per mL <span class="cite">[<a 
href="#XDyallSmith:2003eu">21</a>]</span>. Viruses could be playing a dual
role in these systems; as predators, contributing to the cycling of nutrients
through cell lysis; and as a form of information storage, by providing access to an
auxiliary gene pool that can be utilized by other microorganisms <span class="cite">[<a 
href="#XLindell:2005gz">42</a>,&#x00A0;<a 
href="#XWilliamson:2008vs">94</a>,&#x00A0;<a 
href="#XHurwitz:2013gl">31</a>]</span>.
Studies of viral dynamics in hypersaline systems have showed that they play a
fundamental role in shaping the population structure of microbial communities
<span class="cite">[<a 
href="#XRodriguezBrito:2010in">71</a>,&#x00A0;<a 
href="#XRodriguezValera:2009cr">72</a>]</span>.
<!--l. 75--><p class="indent" >       Microorganisms that live in hypersaline systems require particular physiological
strategies to deal with the high salt concentrations and the potential osmotic pressure
gradient that can generate across the cell membrane. Two different osmotic adaptation
strategies can be found in halophiles: a salt-in strategy, which involves the accumulation
of inorganic ions in the cytoplasm; and a salt-out strategy, that involves pumping ions
out the cell and the accumulation of compatible solutes in the cytoplasm <span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>. The
salt-in strategy is found in the archaeal populations, specifically within the Order
<tspan font-family="cmti" font-size="12">Halobacteriales</tspan>, in Bacteria of the Order <tspan font-family="cmti" font-size="12">Halanerobiales </tspan>and in <tspan font-family="cmti" font-size="12">S. ruber </tspan><span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>. These
organisms accumulate inorganic ions, such as K<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">+</tspan></sup> and Cl<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">-</tspan></sup>, which requires special
                                                                                <img 
src="template70x.png" alt="PICT" >
<img 
src="template71x.png" alt="PICT" >
                                                                                
enzymatic adaptations reected through protein amino acid compositional changes that
favor acidic amino acids, such as glutamate and aspartate. Based on their amino acid
composition profiles, it has been suggested that the uncultured members of the Class
<tspan font-family="cmti" font-size="12">Nanohaloarchaea </tspan>also utilize this strategy. The salt-out strategy is found in
most of the halophilic Bacteria and halophilic methanogenic Archaea <span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>.
These organisms have outward-directed sodium transporters that pump the
Na<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">+</tspan></sup> ions out of the cytoplasm, but more importantly, they accumulate a large
number of organic solutes to mantain the osmotic potential in the cytoplasm
<span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>.
<!--l. 77--><p class="indent" >       Halophilic microorganisms show a diverse suite of metabolic capabilities. Several
dissimilatory metabolic pathways have been described in halophiles (Figure 1.6) across a
wide range of salinity concentrations. The majority of bacterial and archaeal groups
isolated from hypersaline sites are aerobic chemoorganotrophs. In addition, as oxygen
has a low solubility in salt saturated brines <span class="cite">[<a 
href="#XSherwood:1991tg">79</a>]</span>, several anaerobic energy pathways are
available, including electron acceptor substrates such as nitrate, fumarate, and dimethyl
sulfoxide <span class="cite">[<a 
href="#XOren:2008ej">56</a>,&#x00A0;<a 
href="#XOren:2013bc">59</a>]</span>. Primary productivity in saline systems varies according to the
salinity. In moderate saline environments (between 100 to 250 g/L), primary
productivity occurs in microbial mats dominated by members of the <tspan font-family="cmti" font-size="12">Cyanobacteria</tspan>; in
high salt environments, the primary producers are planktonic algae of the genus
<tspan font-family="cmti" font-size="12">Dunaliella </tspan><span class="cite">[<a 
href="#XOren:2009el">57</a>]</span>.
<!--l. 79--><p class="indent" >       Another important characteristic of halophilic organisms, particularly among the
halophilic Archaea, is the presence of microbial rhodopsins, photoactive proteins found
in all three domains of life. Rhodopsins serve as light-driven proton pumps
(bacteriorhodopsin), chloride pumps (halorhodopsin), or phototactic and photophobic
receptors (sensory rhodopsins) <span class="cite">[<a 
href="#XBrown:2013ic">11</a>]</span>. Halorhodopsins and sensory rhodopsins appear to
be limited in their distribution to the <tspan font-family="cmti" font-size="12">Halobacteria</tspan>, with just a few examples found in
other organisms <span class="cite">[<a 
href="#XSharma:2006kn">78</a>]</span>. Chapter 3 describes the discovery a novel type of microbial
rhodopsin, called the xenorhodopsin, which was found in the genome of the
<tspan font-family="cmti" font-size="12">Nanohaloarchaea</tspan>, and that appear to broadly dispered via horizontal gene transfer
between Archaea and Bacteria (although it is not possible yet to establish the
directionality of acquisition).Indeed, the closest homolog to the Nanohaloarchaea
                                                                                <img 
src="template72x.png" alt="PICT" >
<img 
src="template73x.png" alt="PICT" >
                                                                                
xenorhodopsin is a putative sensory rhodopsin found in the cyanobacterium <tspan font-family="cmti" font-size="12">Anabaena</tspan>
sp. PCC 712 <span class="cite">[<a 
href="#XVogeley:2004vh">90</a>,&#x00A0;<a 
href="#XUgalde:2011fw">87</a>]</span>.
<!--l. 81--><p class="indent" >       Even in highly characterized microbial communities <span class="cite">[<a 
href="#XOren:2008ej">56</a>,&#x00A0;<a 
href="#XOren:2012bg">58</a>]</span>], recent studies
using culture independent techniques, such as metagenomics and single-cell genomics,
recovered novel microbial groups <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XGhai:2011hn">28</a>,&#x00A0;<a 
href="#XLopezPerez:2013db">44</a>]</span>. The limited microbial diversity, driven by
extreme salinity systems, makes them ideal model systems to study microbial diversity
using metagenomic methods. We can fully characterize a microbial community
using deep-sequencing approaches, with the goal to reconstruct the genomes
of each member of the community. Through this approach, we identify not
only the functional and phylogenetic diversity of the community, but also the
association of such functions to members of the community. Lastly, it allows
for the study of population genetics within the community, with the goal of
understanding how these organisms adapt and respond to variations in the
environment.
<!--l. 83--><p class="indent" >       It is important to highlight that other saline ecosystems have been studied using
metagenomic approaches. Prior hypersaline metagenomic studies have focused on the
population genomics of single species, such as <tspan font-family="cmti" font-size="12">Haloquadratum walsbyi </tspan><span class="cite">[<a 
href="#XLegault:2006kh">40</a>]</span> and
<tspan font-family="cmti" font-size="12">Salinibacter ruber </tspan><span class="cite">[<a 
href="#XPasic:2009bo">62</a>]</span>, or on the dynamic changes of the microbial and viral diversity
over salinity gradientes and over time <span class="cite">[<a 
href="#XWillner:2009iv">95</a>,&#x00A0;<a 
href="#XRodriguezBrito:2010in">71</a>]</span>. Only recent studies have explored the
microbial and viral diversity of these systems by using assembly-based metagenomics
and single-cell genomics approaches <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XGhai:2011hn">28</a>,&#x00A0;<a 
href="#XGhai:2012fb">27</a>,&#x00A0;<a 
href="#XEmerson:2012gh">23</a>,&#x00A0;<a 
href="#XEmerson:tk">24</a>]</span>.
<!--l. 86--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template74x.png" alt="PICT" >
<img 
src="template75x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template76x.png" alt="PICT" >
<img 
src="template78x.png" alt="PICT" >
                                                                                
<!--l. 88--><p class="noindent" ><img 
src="template77x.png" alt="PIC" class="graphics" width="419.16621pt" height="521.55623pt" ><!--tex4ht:graphics  
name="template77x.png" src="Chapter1/Figures/HaloG22_16S_v2-2.pdf"  
--> <a 
 id="x1-7r2"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.2</tspan>: Phylogenetic tree of <tspan font-family="cmti" font-size="12">Candidatus  </tspan>Halobonum tyrrellensis G22 and
related microorganisms, based on 16S rRNA sequences.
                                                                                <img 
src="template79x.png" alt="PICT" >
<img 
src="template80x.png" alt="PICT" >
                                                                                
<!--l. 91--><p class="indent" >       </div><hr class="endfigure">
<!--l. 94--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template81x.png" alt="PICT" >
<img 
src="template82x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template83x.png" alt="PICT" >
<img 
src="template85x.png" alt="PICT" >
                                                                                
<!--l. 96--><p class="noindent" ><img 
src="template84x.png" alt="PIC" class="graphics" width="419.16621pt" height="300.25703pt" ><!--tex4ht:graphics  
name="template84x.png" src="Chapter1/Figures/HaloG22_v6.pdf"  
--> <a 
 id="x1-8r3"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.3</tspan>: Phylogenetic tree of <tspan font-family="cmti" font-size="12">Candidatus  </tspan>Halobonum tyrrellensis G22 and
related microorganisms, based on phylogenetic marker sequences implemented in
the Phylophlan software <span class="cite">[<a 
href="#XSegata:2013hb">76</a>]</span>
                                                                                <img 
src="template86x.png" alt="PICT" >
<img 
src="template87x.png" alt="PICT" >
                                                                                
<!--l. 99--><p class="indent" >       </div><hr class="endfigure">
<!--l. 102--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template88x.png" alt="PICT" >
<img 
src="template89x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template90x.png" alt="PICT" >
<img 
src="template92x.png" alt="PICT" >
                                                                                
<!--l. 104--><p class="noindent" ><img 
src="template91x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template91x.png" src="Chapter1/Figures/IonicComposition.pdf"  
--> <a 
 id="x1-9r4"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.4</tspan>: Ionic composition of several aquatic systems. Figure from Oren, 2013.
<span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>
                                                                                <img 
src="template93x.png" alt="PICT" >
<img 
src="template94x.png" alt="PICT" >
                                                                                
<!--l. 107--><p class="indent" >       </div><hr class="endfigure">
<!--l. 110--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template95x.png" alt="PICT" >
<img 
src="template96x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template97x.png" alt="PICT" >
<img 
src="template99x.png" alt="PICT" >
                                                                                
<!--l. 112--><p class="noindent" ><img 
src="template98x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template98x.png" src="Chapter1/Figures/HaloMetabolism.pdf"  
--> <a 
 id="x1-10r5"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.5</tspan>:  Salt  concentration  limits  for  some  microbial  metabolic  processes.
Black  bars  indicate  information  based  on  laboratory  studies,  while  white  bars
indicate activities measured in natural microbial communities. Figure from Oren,
2013 <span class="cite">[<a 
href="#XOren:2013bc">59</a>]</span>.
                                                                                <img 
src="template100x.png" alt="PICT" >
<img 
src="template101x.png" alt="PICT" >
                                                                                
<!--l. 115--><p class="indent" >       </div><hr class="endfigure">
                                                                                <img 
src="template102x.png" alt="PICT" >
<img 
src="template103x.png" alt="PICT" >
                                                                                
<!--l. 119--><p class="indent" >       <hr class="float"><div class="float" 
>
                                                                                <img 
src="template104x.png" alt="PICT" >
<img 
src="template105x.png" alt="PICT" >
                                                                                
<a 
 id="x1-11r1"></a>
<tspan font-family="cmbx" font-size="12">Table</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.1</tspan>:  Halophilic  Archaea,  modified  from  <span class="cite">[<a 
href="#XVentosa:2012wo">89</a>]</span>  to  include  the  recently
discovered <tspan font-family="cmti" font-size="12">Nanohaloarchaea </tspan><span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>.
<div class="center" 
>
<!--l. 121--><p class="noindent" >
<!--l. 123--><p class="noindent" > <table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-4-1g"><col 
id="TBL-4-1"><col 
id="TBL-4-2"><col 
id="TBL-4-3"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-4-1-1"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Phylum</tspan>          </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-1-2"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Class</tspan>             </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-1-3"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Genera</tspan>                              </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-2-"><td  style="white-space:wrap; text-align:left;" id="TBL-4-2-1"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmti" font-size="12">Euryarchaeota</tspan>     </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-2-2"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halobacteria</tspan>       </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-2-3"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halobacteria,        Haladaptatus,</tspan>
 <tspan font-family="cmti" font-size="12">Halalkalicoccus,     Halarchaeum,</tspan>
 <tspan font-family="cmti" font-size="12">Haloarcula,           Halobaculum,</tspan>
 <tspan font-family="cmti" font-size="12">Halobiforma,           Halococcus,</tspan>
 <tspan font-family="cmti" font-size="12">Haloferax,       Halogeometricum,</tspan>
 <tspan font-family="cmti" font-size="12">Halogranum,      Halomicrobium,</tspan>
 <tspan font-family="cmti" font-size="12">Halonotius,           Halopelagius,</tspan>
 <tspan font-family="cmti" font-size="12">Halopiger,              Haloplanus,</tspan>
 <tspan font-family="cmti" font-size="12">Haloquadratum,      Halorhabdus,</tspan>
 <tspan font-family="cmti" font-size="12">Halorubrum,           Halorussus,</tspan>
 <tspan font-family="cmti" font-size="12">Halosarcina,          Halosimplex,</tspan>
 <tspan font-family="cmti" font-size="12">Halostagnicola,     Haloterrigena,</tspan>
 <tspan font-family="cmti" font-size="12">Halovivax,                Natrialba,</tspan>
 <tspan font-family="cmti" font-size="12">Natrinema,     Natronoarchaeum,</tspan>
 <tspan font-family="cmti" font-size="12">Natronobacterium,</tspan>
 <tspan font-family="cmti" font-size="12">Natronococcus,</tspan>
 <tspan font-family="cmti" font-size="12">Natronolimnobius,</tspan>
 <tspan font-family="cmti" font-size="12">Natronomonas,   Natronorubrum,</tspan>
 <tspan font-family="cmti" font-size="12">Salarchaeum</tspan>                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-3-"><td  style="white-space:wrap; text-align:left;" id="TBL-4-3-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-4-"><td  style="white-space:wrap; text-align:left;" id="TBL-4-4-1"  
class="td11"> <!--l. 134--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-4-2"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmti" font-size="12">Methanomicrobia</tspan> </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-4-3"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmti" font-size="12">Methanohalobium,</tspan>
 <tspan font-family="cmti" font-size="12">Methanocalculus,</tspan>
 <tspan font-family="cmti" font-size="12">Methanohalohilus,</tspan>
 <tspan font-family="cmti" font-size="12">Methanosalsum</tspan>                       </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-5-"><td  style="white-space:wrap; text-align:left;" id="TBL-4-5-1"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmti" font-size="12">Nanohaloarchaea</tspan>  </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-5-2"  
class="td11"> <!--l. 134--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-5-3"  
class="td11"> <!--l. 134--><p class="noindent" ><tspan font-family="cmti" font-size="12">Nanosalina, Nanosalinarum</tspan>        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-4-6-"><td  style="white-space:wrap; text-align:left;" id="TBL-4-6-1"  
class="td11"> <!--l. 134--><p class="noindent" >               </td></tr></table></div>
                                                                                <img 
src="template106x.png" alt="PICT" >
<img 
src="template107x.png" alt="PICT" >
                                                                                
       </div><hr class="endfloat" />
<div class="center" 
>
<!--l. 144--><p class="noindent" >
<a 
 id="x1-12r2"></a><!--l. 147--><div class="longtable"> <table id="TBL-5" class="longtable" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-5-1g"><col 
id="TBL-5-1"><col 
id="TBL-5-2"><col 
id="TBL-5-3"></colgroup>
<tr  
 style="vertical-align:baseline;" id="TBL-5-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-1-1"  
class="td11">
 <!--l. 148--><p class="noindent" >               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-2-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-2-1"  
class="td11"></td>           <div class="multicolumn"  style="white-space:nowrap; text-align:center;"><tspan font-family="cmbx" font-size="12">Table</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.2</tspan>Halophilic Bacteria, modified from <span class="cite">[<a 
href="#XVentosa:2012wo">89</a>]</span>.       </div>                  <a 
 id="x1-13"></a>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-3-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-3-1"  
class="td11">
 <!--l. 151--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Phylum</tspan>          </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-3-2"  
class="td11">
 <!--l. 151--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Class</tspan>                       </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-3-3"  
class="td11">
 <!--l. 151--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Genera</tspan>                           </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-4-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-4-1"  
class="td11">
 <!--l. 153--><p class="noindent" >               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-5-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-5-1"  
class="td11">                   </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-5-2"  
class="td11">                           </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-5-3"  
class="td11"></td></tr>
<tr  
 style="vertical-align:baseline;" id="TBL-5-15-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-15-1"  
class="td11">
 <!--l. 168--><p class="noindent" ><tspan font-family="cmti" font-size="12">Actinobacteria</tspan>    </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-15-2"  
class="td11">
 <!--l. 168--><p class="noindent" ><tspan font-family="cmti" font-size="12">Actinobacteria</tspan>              </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-15-3"  
class="td11">
 <!--l. 168--><p class="noindent" ><tspan font-family="cmti" font-size="12">Actinopolyspora,</tspan>
 <tspan font-family="cmti" font-size="12">Amycolatopsis,</tspan>
 <tspan font-family="cmti" font-size="12">Georgenia,   Corynebacterium,</tspan>
 <tspan font-family="cmti" font-size="12">Haloactinobacterium,</tspan>
 <tspan font-family="cmti" font-size="12">Haloactinopolyspora,</tspan>
 <tspan font-family="cmti" font-size="12">Haloechinothrix,</tspan>
 <tspan font-family="cmti" font-size="12">Haloglycomyces,</tspan>
 <tspan font-family="cmti" font-size="12">Nesterenjonia,</tspan>
 <tspan font-family="cmti" font-size="12">Nocardiopsis, Haloactinospora,</tspan>
 <tspan font-family="cmti" font-size="12">Streptomonospora,</tspan>
 <tspan font-family="cmti" font-size="12">Isoptericola,        Prauserella,</tspan>
 <tspan font-family="cmti" font-size="12">Saccharomonospora,</tspan>
 <tspan font-family="cmti" font-size="12">Saccharopolyspora</tspan>                 </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-16-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-16-1"  
class="td11">
 <!--l. 171--><p class="noindent" ><tspan font-family="cmti" font-size="12">Bacteroidetes</tspan>     </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-16-2"  
class="td11">
 <!--l. 171--><p class="noindent" ><tspan font-family="cmti" font-size="12">Bacteroidia</tspan>                  </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-16-3"  
class="td11">
 <!--l. 171--><p class="noindent" ><tspan font-family="cmti" font-size="12">Anaerophaga</tspan>                       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-17-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-17-1"  
class="td11">
                                                                                <img 
src="template108x.png" alt="PICT" >
<img 
src="template109x.png" alt="PICT" >
                                                                                
</td></tr><tr 
class="cline"><td></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-18-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-18-1"  
class="td11">
 <!--l. 174--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-18-2"  
class="td11">
 <!--l. 174--><p class="noindent" ><tspan font-family="cmti" font-size="12">Flavobacteria</tspan>                </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-18-3"  
class="td11">
 <!--l. 174--><p class="noindent" ><tspan font-family="cmti" font-size="12">Gramella, Psychroflexus</tspan>         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-19-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-19-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-20-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-20-1"  
class="td11">
 <!--l. 176--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-20-2"  
class="td11">
 <!--l. 176--><p class="noindent" ><tspan font-family="cmti" font-size="12">Sphingobacteria</tspan>             </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-20-3"  
class="td11">
 <!--l. 176--><p class="noindent" ><tspan font-family="cmti" font-size="12">Salinibacter, Salisaeta</tspan>            </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-21-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-21-1"  
class="td11">
 <!--l. 179--><p class="noindent" ><tspan font-family="cmti" font-size="12">Cyanobacteria</tspan>    </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-21-2"  
class="td11">
 <!--l. 179--><p class="noindent" >                      </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-21-3"  
class="td11">
 <!--l. 179--><p class="noindent" ><tspan font-family="cmti" font-size="12">Rubidibacter,  Prochlorococcus,</tspan>
 <tspan font-family="cmti" font-size="12">Halospirulina</tspan>                      </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-22-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-22-1"  
class="td11">
 <!--l. 182--><p class="noindent" ><tspan font-family="cmti" font-size="12">Firmicutes</tspan>         </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-22-2"  
class="td11">
 <!--l. 182--><p class="noindent" ><tspan font-family="cmti" font-size="12">Bacilli</tspan>                        </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-22-3"  
class="td11">
 <!--l. 182--><p class="noindent" ><tspan font-family="cmti" font-size="12">Alkalibacillus, Aquisalibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Bacillus,            Filobacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Gracilibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Halalkalibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Halolactibacillus, Halobacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Jeotgalibacillus,  Lentibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Oceanobacillus,Ornithinibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Paraliobacillus,   Piscibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Pontibacillus,   Salimicrobium,</tspan>
 <tspan font-family="cmti" font-size="12">Salinibacillus,      Salirhabdus,</tspan>
 <tspan font-family="cmti" font-size="12">Salsuginibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Sediminibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Salinicoccus,     Tenuibacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Thalassobacillus, Virgibacillus</tspan>  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-23-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-23-1"  
class="td11">
                                                                                <img 
src="template110x.png" alt="PICT" >
<img 
src="template111x.png" alt="PICT" >
                                                                                
</td></tr><tr 
class="cline"><td></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-24-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-24-1"  
class="td11">
 <!--l. 184--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-24-2"  
class="td11">
 <!--l. 184--><p class="noindent" ><tspan font-family="cmti" font-size="12">Clostridia</tspan>                    </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-24-3"  
class="td11">
 <!--l. 184--><p class="noindent" ><tspan font-family="cmti" font-size="12">Acetohalobium,</tspan>
 <tspan font-family="cmti" font-size="12">Halanaerobacter,</tspan>
 <tspan font-family="cmti" font-size="12">Halanaerobium,</tspan>
 <tspan font-family="cmti" font-size="12">Halobacteroides,      Halocella,</tspan>
 <tspan font-family="cmti" font-size="12">Halonatronum,</tspan>
 <tspan font-family="cmti" font-size="12">Halothermothrix,</tspan>
 <tspan font-family="cmti" font-size="12">Natranaerobius,   Natrionella,</tspan>
 <tspan font-family="cmti" font-size="12">Natronovirga,</tspan>
 <tspan font-family="cmti" font-size="12">Orenia, Selenihalanaerobacter,</tspan>
 <tspan font-family="cmti" font-size="12">Sporohalobacter</tspan>                    </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-25-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-25-1"  
class="td11">
 <!--l. 187--><p class="noindent" ><tspan font-family="cmti" font-size="12">Proteobacteria</tspan>    </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-25-2"  
class="td11">
 <!--l. 187--><p class="noindent" ><tspan font-family="cmti" font-size="12">Alphaproteobacteria</tspan>        </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-25-3"  
class="td11">
 <!--l. 187--><p class="noindent" ><tspan font-family="cmti" font-size="12">Antarctobacter,   Citreimonas,</tspan>
 <tspan font-family="cmti" font-size="12">Dichotomicrobium,</tspan>
 <tspan font-family="cmti" font-size="12">Fodinicurvata, Hwanghaeicola,</tspan>
 <tspan font-family="cmti" font-size="12">Hyphomonas,      Jannaschia,</tspan>
 <tspan font-family="cmti" font-size="12">Maribaculum,         Maribius,</tspan>
 <tspan font-family="cmti" font-size="12">Marispirillum,   Methylarcula,</tspan>
 <tspan font-family="cmti" font-size="12">Oceanibulbus,       Oceanicola,</tspan>
 <tspan font-family="cmti" font-size="12">Palleronia,          Paracoccus,</tspan>
 <tspan font-family="cmti" font-size="12">Ponticoccus,        Rhodobium,</tspan>
 <tspan font-family="cmti" font-size="12">Rhodotalassium,  Rhodovibrio,</tspan>
 <tspan font-family="cmti" font-size="12">Rhodovulum,     Roseicitreum,</tspan>
 <tspan font-family="cmti" font-size="12">Roseinatronobacter,</tspan>
 <tspan font-family="cmti" font-size="12">Roseisalinus,        Roseospira,</tspan>
 <tspan font-family="cmti" font-size="12">Roseovarius,    Salinihabitans,</tspan>
 <tspan font-family="cmti" font-size="12">Salipiger,       Sediminimonas,</tspan>
 <tspan font-family="cmti" font-size="12">Shimia,            Sulfitobacter,</tspan>
 <tspan font-family="cmti" font-size="12">Tropicibacter,      Woodsholea,</tspan>
 <tspan font-family="cmti" font-size="12">Yangia</tspan>                              </td>
                                                                                <img 
src="template112x.png" alt="PICT" >
<img 
src="template113x.png" alt="PICT" >
                                                                                
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-26-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-26-1"  
class="td11">
                                                                                <img 
src="template114x.png" alt="PICT" >
<img 
src="template115x.png" alt="PICT" >
                                                                                
</td></tr><tr 
class="cline"><td></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-27-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-27-1"  
class="td11">
 <!--l. 189--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-27-2"  
class="td11">
 <!--l. 189--><p class="noindent" ><tspan font-family="cmti" font-size="12">Gammaproteobacteria</tspan>     </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-27-3"  
class="td11">
 <!--l. 189--><p class="noindent" ><tspan font-family="cmti" font-size="12">Aidingimonas,     Alcanivorax,</tspan>
 <tspan font-family="cmti" font-size="12">Alkalilmnicola,   Alteromonas,</tspan>
 <tspan font-family="cmti" font-size="12">Aestuariibacter,</tspan>
 <tspan font-family="cmti" font-size="12">Aquisalimonas,</tspan>
 <tspan font-family="cmti" font-size="12">Arhodomonas,    Carnimonas,</tspan>
 <tspan font-family="cmti" font-size="12">Chromohalobacter,</tspan>
 <tspan font-family="cmti" font-size="12">Cobetia,     Ectothiorhodspira,</tspan>
 <tspan font-family="cmti" font-size="12">Ectothiorhodosinus,</tspan>
 <tspan font-family="cmti" font-size="12">Glaciecola,       Gilvimarinus,</tspan>
 <tspan font-family="cmti" font-size="12">Haliea,        Halochromatium,</tspan>
 <tspan font-family="cmti" font-size="12">Halomonas,    Halorhodospira,</tspan>
 <tspan font-family="cmti" font-size="12">Halospina,    Halothiobacillus,</tspan>
 <tspan font-family="cmti" font-size="12">Idiomarina,         Kushneria,</tspan>
 <tspan font-family="cmti" font-size="12">Marichromatium,</tspan>
 <tspan font-family="cmti" font-size="12">Marinobacter,</tspan>
 <tspan font-family="cmti" font-size="12">Marinobacterium,      Melitea,</tspan>
 <tspan font-family="cmti" font-size="12">Methylohalomonas,</tspan>
 <tspan font-family="cmti" font-size="12">Microbulbifer,</tspan>
 <tspan font-family="cmti" font-size="12">Modicisalibacter,    Nitrincola,</tspan>
 <tspan font-family="cmti" font-size="12">Oleispira,    Pseudidiomarina,</tspan>
 <tspan font-family="cmti" font-size="12">Pseudoaltermonas,</tspan>
 <tspan font-family="cmti" font-size="12">Psychromonas,  Pseudomonas,</tspan>
 <tspan font-family="cmti" font-size="12">Saccarospirillum,      Salicola,</tspan>
 <tspan font-family="cmti" font-size="12">Salinicola,        Salinisphaera,</tspan>
 <tspan font-family="cmti" font-size="12">Salinivibrio,  Thioalkalibacter,</tspan>
 <tspan font-family="cmti" font-size="12">Thioalkalivibrio,</tspan>
 <tspan font-family="cmti" font-size="12">Thiohalobacter,</tspan>
 <tspan font-family="cmti" font-size="12">Thiohalorhabdus,</tspan>
 <tspan font-family="cmti" font-size="12">Thiohalocapsa,</tspan>
 <tspan font-family="cmti" font-size="12">Thiohalomonas,</tspan>
 <tspan font-family="cmti" font-size="12">Thiohalophilus, Thiohalospira,</tspan>
 <tspan font-family="cmti" font-size="12">Thiomicrospira</tspan>                    </td>
                                                                                <img 
src="template116x.png" alt="PICT" >
<img 
src="template117x.png" alt="PICT" >
                                                                                
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-28-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-28-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-29-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-29-1"  
class="td11">
 <!--l. 191--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-29-2"  
class="td11">
 <!--l. 191--><p class="noindent" ><tspan font-family="cmti" font-size="12">Deltaproteobacteria</tspan>         </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-29-3"  
class="td11">
 <!--l. 191--><p class="noindent" ><tspan font-family="cmti" font-size="12">Desulfocella, Desulfohalobium,</tspan>
 <tspan font-family="cmti" font-size="12">Desulfonatronospira,</tspan>
 <tspan font-family="cmti" font-size="12">Desulfosalsimonas,</tspan>
 <tspan font-family="cmti" font-size="12">Desulfovermiculus,</tspan>
 <tspan font-family="cmti" font-size="12">Desulfovibrio, Desulfurivibrio</tspan>   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-30-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-30-1"  
class="td11">
</td></tr><tr 
class="cline"><td></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-31-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-31-1"  
class="td11">
 <!--l. 193--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-31-2"  
class="td11">
 <!--l. 193--><p class="noindent" ><tspan font-family="cmti" font-size="12">Epsilonproteobacteria</tspan>      </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-31-3"  
class="td11">
 <!--l. 193--><p class="noindent" ><tspan font-family="cmti" font-size="12">Arcobacter,       Sulfurimonas,</tspan>
 <tspan font-family="cmti" font-size="12">Sulfurovum</tspan>                         </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-32-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-32-1"  
class="td11">
 <!--l. 196--><p class="noindent" ><tspan font-family="cmti" font-size="12">Spirochaetes</tspan>       </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-32-2"  
class="td11">
 <!--l. 196--><p class="noindent" ><tspan font-family="cmti" font-size="12">Spirochaetes</tspan>                 </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-32-3"  
class="td11">
 <!--l. 196--><p class="noindent" ><tspan font-family="cmti" font-size="12">Spirochaeta</tspan>                         </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-33-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-33-1"  
class="td11">
 <!--l. 199--><p class="noindent" ><tspan font-family="cmti" font-size="12">Tenericutes</tspan>        </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-33-2"  
class="td11">
 <!--l. 199--><p class="noindent" ><tspan font-family="cmti" font-size="12">Mollicutes</tspan>                   </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-33-3"  
class="td11">
 <!--l. 199--><p class="noindent" ><tspan font-family="cmti" font-size="12">Haloplasma</tspan>                         </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-34-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-34-1"  
class="td11">
 <!--l. 202--><p class="noindent" ><tspan font-family="cmti" font-size="12">Thermotogae</tspan>      </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-34-2"  
class="td11">
 <!--l. 202--><p class="noindent" ><tspan font-family="cmti" font-size="12">Thermotogae</tspan>                </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-34-3"  
class="td11">
 <!--l. 202--><p class="noindent" ><tspan font-family="cmti" font-size="12">Petrotoga</tspan>                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-35-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-35-1"  
class="td11">                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-36-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-36-1"  
class="td11">                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-37-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-37-1"  
class="td11">
 <!--l. 204--><p class="noindent" >               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-38-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-38-1"  
class="td11">
 <!--l. 204--><p class="noindent" >               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-39-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-39-1"  
class="td11">
 <!--l. 204--><p class="noindent" >               </td>
</tr>
<tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-13-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-13-1"  
class="td11">
 <!--l. 165--><p class="noindent" >               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-14-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-14-1"  
class="td11">
 <!--l. 165--><p class="noindent" >               </td>
</tr>
</table></div>
</div>
<a 
 id="x1-14r3"></a>
<!--l. 212--><p class="nopar" ><tspan font-family="cmbx" font-size="12">1.3</tspan>    <tspan font-family="cmbx" font-size="12">Lake Tyrrell, Australia, as a model ecosystem</tspan>
<a 
 id="Q1-1-7"></a>
                                                                                <img 
src="template118x.png" alt="PICT" >
<img 
src="template119x.png" alt="PICT" >
                                                                                
<!--l. 214--><p class="indent" >       Lake Tyrrell, Victoria, Australia, is located in the center of the Murray
Basin Plains (Figure 1.7), in a region with a semi-arid climate, average rainfall
of 300 mm/year, and an evaporation rate of &#x00A0;2000 mm/year <span class="cite">[<a 
href="#XMacumber:1992ty">45</a>]</span>. The lake
is considered an acid-hypersaline system, where low-pH, oxygenated, saline,
metal-rich groundwater from springs is evapo-concentrated and mixed with
near-neutral pH waters, rich in sulfides <span class="cite">[<a 
href="#XLong:1992ie">43</a>]</span>. The lake shows seasonal salinity
variations. During winter, the salt content is approximately <tspan font-family="cmmi" font-size="12">&#x003E;</tspan>250 g/L; in summer,
due to water evaporation, the residual brines reach concentrations <tspan font-family="cmmi" font-size="12">&#x003E;</tspan>330 g/L
<span class="cite">[<a 
href="#XMacumber:1992ty">45</a>]</span>.
<!--l. 216--><p class="indent" >       Lake Tyrrell has been described and studied in detail in terms of its hydrological
and geochemical features <span class="cite">[<a 
href="#XLong:1992ie">43</a>,&#x00A0;<a 
href="#XMacumber:1992ty">45</a>,&#x00A0;<a 
href="#XJones:2006jn">33</a>]</span>], making it a great candidate for microbiological
characterization. Recent projects have used a diverse array of microbiological techniques
to study the microbial diversity of Lake Tyrrell, including Eukaryotes <span class="cite">[<a 
href="#XKarlaBHeidelberg:2013jc">37</a>]</span>, Archaea
and Bacteria <span class="cite">[<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XUgalde:2013hb">86</a>]</span>, and Viruses <span class="cite">[<a 
href="#XEmerson:2012gh">23</a>,&#x00A0;<a 
href="#XEmerson:tk">24</a>]</span>. Future work will combine the
metagenomic, proteomic, and available geochemical information to provide a more
integrative description of the interactions between microbes, viruses, and the
environment.
<!--l. 218--><p class="indent" >       In this thesis, we explore the microbial diversity of the Lake Tyrrell ecosystem,
based on the data generated through a metagenomic study. In particular, our study
highlights how, by assembling metagenomic data, we obtain a more complete picture of
the microbial diversity present in the community, and also how we can exploit this
information to obtain a broad picture of the phylogenetic and functional diversity,
and to explore in detail the genetic diversity of the members of the microbial
community.
<!--l. 220--><p class="indent" >       <tspan font-family="cmbx" font-size="12">Chapter 2 </tspan>describes how the assembly of metagenomic datasets allowed the
recovery and identification of novel microbial groups that are abundant in the
hypersaline waters of Lake Tyrrell, and other hypersaline ecosystems. Using additional
metadata, such as size fractionation, sequence nucleotide composition, and
phylogenetic binning, two near-complete genomes from a novel Class of Archaea, the
<tspan font-family="cmti" font-size="12">Nanohaloarchaea</tspan>, were recovered from the metagenomic samples. . This chapter
represents work that has already been published <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>. My role, in this study was in the
                                                                                <img 
src="template120x.png" alt="PICT" >
<img 
src="template121x.png" alt="PICT" >
                                                                                
classification of sequences into phylogenetic groups using statistical approaches, such as
non-metric multidimensional scaling, and exploring the functional diversity of the two
novel genomes.
<!--l. 222--><p class="indent" >       <tspan font-family="cmbx" font-size="12">Chapter 3 </tspan>corresponds to the bioinformatic analysis and description of a novel
type of microbial rhodopsin, xenorhodopsin, identified from the genomes of the
Nanohaloarchaea.The results convey how this rhodopsin appears to be a new class of
microbial rhodopsin based on phylogenetic analysis and on the presence of unique amino
acid signatures. This work has already been published <span class="cite">[<a 
href="#XUgalde:2011fw">87</a>]</span>, where I was the lead author
of the study.
<!--l. 224--><p class="indent" >       <tspan font-family="cmbx" font-size="12">Chapter 4 </tspan>describes the assembly of several genomes from the Lake
Tyrrell metagenome, based on the combination of assembly-based approaches
and metadata. These results provide a framework for future analyses of this
ecosystem, providing a set of habitat-specific genomes, including phylogenetic,
functional, and genetic diversity. This work is already published <span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>[?]. My role
in this study was developing the methods for classification of the assembled
scaffolds into different phylogenetic groups and developing novel visualization
and analysis approaches to compare the functional repertoire of community
members.
<!--l. 226--><p class="indent" >       <tspan font-family="cmbx" font-size="12">Chapter 5</tspan>, leverages the assembled habitat-specific genomes, and describes a
bioinformatic approach for the analysis of genetic diversity in a metagenome sample.
Using this framework, a deep-sequencing approach was used to characterize the
genetic heterogeneity present in the microbial members of the Lake Tyrrell
community.
<!--l. 229--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template122x.png" alt="PICT" >
<img 
src="template123x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template124x.png" alt="PICT" >
<img 
src="template126x.png" alt="PICT" >
                                                                                
<!--l. 231--><p class="noindent" ><img 
src="template125x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template125x.png" src="Chapter1/Figures/LT_Map.pdf"  
--> <a 
 id="x1-15r6"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;1.6</tspan>: Location of Lake Tyrrell in the southeastern region of Australia.
Figure from Macumber, 1992. <span class="cite">[<a 
href="#XMacumber:1992ty">45</a>]</span>.
                                                                                <img 
src="template127x.png" alt="PICT" >
<img 
src="template128x.png" alt="PICT" >
                                                                                
<!--l. 234--><p class="indent" >       </div><hr class="endfigure">
                                                                                <img 
src="template129x.png" alt="PICT" >
<img 
src="template130x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template131x.png" alt="PICT" >
<img 
src="template132x.png" alt="PICT" >
                                                                                
<a 
 id="x1-16r2"></a>
<a 
 id="Q1-1-8"></a>
<!--l. 216--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Chapter</tspan>
<tspan font-family="cmbx" font-size="12">2</tspan>
<!--l. 216--><p class="noindent" ><tspan font-family="cmbxti" font-size="10">De</tspan>
<tspan font-family="cmbxti" font-size="10">novo</tspan>
<tspan font-family="cmbx" font-size="12">metagenomic</tspan>
<tspan font-family="cmbx" font-size="12">assembly</tspan>
<tspan font-family="cmbx" font-size="12">reveals</tspan>
<tspan font-family="cmbx" font-size="12">abundant</tspan>
<tspan font-family="cmbx" font-size="12">novel</tspan>
<tspan font-family="cmbx" font-size="12">major</tspan>
<tspan font-family="cmbx" font-size="12">lineage</tspan>
<tspan font-family="cmbx" font-size="12">of</tspan>
                                                                                <img 
src="template133x.png" alt="PICT" >
<img 
src="template134x.png" alt="PICT" >
                                                                                
<tspan font-family="cmbx" font-size="12">Archaea</tspan>
<tspan font-family="cmbx" font-size="12">in</tspan>
<tspan font-family="cmbx" font-size="12">hypersaline</tspan>
<tspan font-family="cmbx" font-size="12">communities</tspan>
<!--l. 218--><p class="indent" >
                                                                                <img 
src="template135x.png" alt="PICT" >
<img 
src="template136x.png" alt="PICT" >
                                                                                
<!--l. 218--><p class="indent" >
                                                                                <img 
src="template137x.png" alt="PICT" >
<img 
src="template138x.png" alt="PICT" >
                                                                                
<!--l. 219--><p class="indent" >       Chapter 2 is a full reprint of: De novo metagenomic assembly reveals abundant
novel major lineage of Archaea in hypersaline communnities. P. Narasingarao, S. Podell,
J.A. Ugalde, C. Brochier-Armanet, J.B. Emerson, J.J. Brocks, K.B. Heidelberg, J.F.
Banfield and E.E. Allen. <tspan font-family="cmti" font-size="12">ISME Journal</tspan>, <tspan font-family="cmbx" font-size="12">6</tspan>,81-93. 2012 (doi: 10.1038/ismej.2011.78), with
permission from all coauthors.
                                                                                <img 
src="template139x.png" alt="PICT" >
<img 
src="template140x.png" alt="PICT" >
                                                                                
<a 
 id="x1-17r3"></a>
<a 
 id="Q1-1-8"></a>
<!--l. 223--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Chapter</tspan>
<tspan font-family="cmbx" font-size="12">3</tspan>
<!--l. 223--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Xenorhodopsins,</tspan>
<tspan font-family="cmbx" font-size="12">an</tspan>
<tspan font-family="cmbx" font-size="12">enigmatic</tspan>
<tspan font-family="cmbx" font-size="12">new</tspan>
<tspan font-family="cmbx" font-size="12">class</tspan>
<tspan font-family="cmbx" font-size="12">of</tspan>
<tspan font-family="cmbx" font-size="12">microbial</tspan>
<tspan font-family="cmbx" font-size="12">rhodopsins</tspan>
<tspan font-family="cmbx" font-size="12">horizontally</tspan>
<tspan font-family="cmbx" font-size="12">transferred</tspan>
                                                                                <img 
src="template141x.png" alt="PICT" >
<img 
src="template142x.png" alt="PICT" >
                                                                                
<tspan font-family="cmbx" font-size="12">between</tspan>
<tspan font-family="cmbx" font-size="12">Archaea</tspan>
<tspan font-family="cmbx" font-size="12">and</tspan>
<tspan font-family="cmbx" font-size="12">Bacteria</tspan>
<!--l. 225--><p class="indent" >
                                                                                <img 
src="template143x.png" alt="PICT" >
<img 
src="template144x.png" alt="PICT" >
                                                                                
<!--l. 225--><p class="indent" >
                                                                                <img 
src="template145x.png" alt="PICT" >
<img 
src="template146x.png" alt="PICT" >
                                                                                
<!--l. 226--><p class="indent" >       Chapter 3 is a full reprint of: Xenorhodopsins, an enigmatic new class of
microbial rhodopsins horizontally transferred between archaea and bacteria. J.A.
Ugalde, S. Podell, P. Narasingarao and E.E. Allen. <tspan font-family="cmti" font-size="12">Biology Direct</tspan>, <tspan font-family="cmbx" font-size="12">6</tspan>,52. 2011 (doi:
10.1186/1745-6150-6-52), with permission from all coauthors.
                                                                                <img 
src="template147x.png" alt="PICT" >
<img 
src="template148x.png" alt="PICT" >
                                                                                
<a 
 id="x1-18r4"></a>
<a 
 id="Q1-1-8"></a>
<!--l. 230--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Chapter</tspan>
<tspan font-family="cmbx" font-size="12">4</tspan>
<!--l. 230--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Assembly-driven</tspan>
<tspan font-family="cmbx" font-size="12">community</tspan>
<tspan font-family="cmbx" font-size="12">genomics</tspan>
<tspan font-family="cmbx" font-size="12">of</tspan>
<tspan font-family="cmbx" font-size="12">a</tspan>
<tspan font-family="cmbx" font-size="12">hypersaline</tspan>
<tspan font-family="cmbx" font-size="12">microbial</tspan>
<tspan font-family="cmbx" font-size="12">ecosystem</tspan>
<!--l. 232--><p class="indent" >
                                                                                <img 
src="template149x.png" alt="PICT" >
<img 
src="template150x.png" alt="PICT" >
                                                                                
<!--l. 232--><p class="indent" >
                                                                                <img 
src="template151x.png" alt="PICT" >
<img 
src="template152x.png" alt="PICT" >
                                                                                
<!--l. 233--><p class="indent" >       Chapter 4 is a full reprint of: Assembly-driven community genomics of a
hypersaline microbial ecosystem. S. Podell, J.A. Ugalde, P. Narasingarao, J.F.
Banfield, K.B. Heidelberg and E.E. Allen. <tspan font-family="cmti" font-size="12">PLoS One</tspan>, <tspan font-family="cmbx" font-size="12">8</tspan>4:e61692. 2013 (doi:
10.1371/journal.pone.0061692), with permission from all coauthors.
                                                                                <img 
src="template153x.png" alt="PICT" >
<img 
src="template154x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template155x.png" alt="PICT" >
<img 
src="template156x.png" alt="PICT" >
                                                                                
<a 
 id="x1-19r5"></a>
<a 
 id="Q1-1-8"></a>
<!--l. 1--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Chapter</tspan>
<tspan font-family="cmbx" font-size="12">5</tspan>
<!--l. 1--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Microbial</tspan>
<tspan font-family="cmbx" font-size="12">population</tspan>
<tspan font-family="cmbx" font-size="12">structure</tspan>
<tspan font-family="cmbx" font-size="12">and</tspan>
<tspan font-family="cmbx" font-size="12">genetic</tspan>
<tspan font-family="cmbx" font-size="12">heterogeneity</tspan>
<tspan font-family="cmbx" font-size="12">in</tspan>
<tspan font-family="cmbx" font-size="12">a</tspan>
<tspan font-family="cmbx" font-size="12">hypersaline</tspan>
                                                                                <img 
src="template157x.png" alt="PICT" >
<img 
src="template158x.png" alt="PICT" >
                                                                                
<tspan font-family="cmbx" font-size="12">environment</tspan>
<a 
 id="x1-20r1"></a>
       <tspan font-family="cmbx" font-size="12">5.1</tspan>    <tspan font-family="cmbx" font-size="12">Abstract</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 5--><p class="nopar" >
<a 
 id="x1-21r2"></a>
<!--l. 5--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.2</tspan>    <tspan font-family="cmbx" font-size="12">Introduction</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 7--><p class="indent" >       The increasing number of microbial genomes being sequenced over the last few
years, due to the drastic price reduction on the cost of sequencing a genome, has allow
for comparisons among the genomes of closelry related microbial species and even more
important strains from the same microbial species (REF). These comparisons have
showed that even closely related strains of microorganisms, have enough differences in
their genomic sequences that could explain their different functional behaivor
(REF), and is a reflection of the different evolutionary histories of each strain
(REF). This has been studied in detail specially in medical applications, where .
[Multiple studies and genomes, selective pressure]. In non-medical studies, a recent
studie by (Science, shapiro), studied multiple Vibrio genomes, comparison,
etc.
<!--l. 10--><p class="indent" >       With the development of community genomics approaches, we have knwo started
to comprehend the level of genetic variation that can be found in natural microbial
population (multiple REFs). In particular, the use of assembly-based approaches, allows
us to reconstruct genomes (consensus o poulation genomes) from the members of the
microbial community under study, and then using bioinformatic approaches, we can
assest the level of genetic diversity that is present in a microbial population (REF).
Most of the current studies have focus on very simple systems (REF, ISMEJ,
enrichemnts) or only on a few members of the population under study (REF, Jill). In
                                                                                <img 
src="template159x.png" alt="PICT" >
<img 
src="template160x.png" alt="PICT" >
                                                                                
contrast, one of the studies that used a great amount of available data was based on
metagenomic sequenecs obtained from the human microbiome, but in this
case the genomes used were not necessary obtained from the same samples
where the metagenomic sequences were obtaiend. In this work, we used as a
foundation the available information that was recovered from a assembly-based
metagenomic study in the Lake Tyrreell hypersaline ecosystem, using the information
as a reference, and then using deep-sequencing approaches, using Illumina
sequencing, to assest and study the level of genetic variation that is present in a
natural microbial community. Based on this information we can assest the
level of genetic variation between populations, and also start to comprehend
the effect of natural selection in these populations. (REFs, check microbiome
paper).
<!--l. 13--><p class="indent" >       we can observe the current status of the community, and ask questions later by
sampling other times (establish a baseline for future studies).
<!--l. 15--><p class="indent" >
<a 
 id="x1-22r3"></a>
<!--l. 15--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.3</tspan>    <tspan font-family="cmbx" font-size="12">Material and Methods</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 17--><p class="indent" >
<a 
 id="x1-23r1"></a>
<!--l. 17--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.3.1</tspan>   <tspan font-family="cmbx" font-size="12">Sample collection and sequencing</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 18--><p class="indent" >       Surface water samples from Lake Tyrrell were collected in 2007, at two different
seasons, summer (January) and winder (August), with with two days of difference in
each season (January 23 and 25, August 7 and 9). Each water sample was filtered
directly into a Sterivex cartridge (Milipore, Bedford, MA, USA) (0.22 <tspan font-family="tcrm" font-size="1200">µ</tspan>m) using a
peristaltic pump. For DNA extraction each Sterivex was processed according to the
following protocol:
      <ul class="itemize1">
                                                                                <img 
src="template161x.png" alt="PICT" >
<img 
src="template162x.png" alt="PICT" >
                                                                                
      <li class="itemize">Addition of Proteinase K to a final concentration of 0.5 mg/ml<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">-1</tspan></sup> and SDS
      to a final concentration of 1%.
      </li>
      <li class="itemize">Incubation at 55<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">o</tspan></sup>C for 25 minutes, followed by incubation at 70<sup class="textsuperscript"><tspan font-family="cmr" font-size="10">o</tspan></sup>C for 5
      minutes.
      </li>
      <li class="itemize">Transfer of the lysate from the Sterivex, to a clean Eppendorf tube.
      </li>
      <li class="itemize">Nucleic acid extraction with two steps of phenol:chloroform extraction.</li></ul>
<!--l. 26--><p class="indent" >       For all four samples, the sequencing library construction was performed by the
UC San Diego IGM Genomics Center. The four libraries were multiplexed and
sequenced on a single HiSeq lane (Illumina, San Diego, CA), on high-throughput mode
(2X100 PE reads).
<!--l. 28--><p class="indent" >       The demultiplexed reads were processed using Nesoni 0.117
(http://www.vicbioinformatics.com/software.nesoni.shtml), to remove adapters, trim
low quality positions and remove low quality reads from the datasets. For trimming a
minimum quality score of 20 was used, and all reads shorter than 70 nucleotides (after
trimming) were removed.
<!--l. 30--><p class="indent" >
<a 
 id="x1-24r2"></a>
<!--l. 30--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.3.2</tspan>   <tspan font-family="cmbx" font-size="12">Read mapping</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 32--><p class="indent" >       The trimmed reads were mapped against a set of habitat-specific genomes (Table
5.1 generated by the assembly of metagenomic information of the Lake Tyrrell microbial
community <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XPodell:2013fp">66</a>]</span>. In addition, an archaeal isolate, <tspan font-family="cmti" font-size="12">Candidatus </tspan>Halobonum
tyrrellensis <span class="cite">[<a 
href="#XUgalde:2013hb">86</a>]</span>, obtained from samples collected in August of 2007, was also included in
the analysis. Each sample was mapped independently to the set of reference
genomes, using Bowtie 2.2.1 <span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span> with the <tspan font-family="cmti" font-size="12">very-sensitive </tspan>alignment option, and
adjusting the N-ceiling function to 0,0.01 to reduce the number of ambiguous
                                                                                <img 
src="template163x.png" alt="PICT" >
<img 
src="template165x.png" alt="PICT" >
                                                                                
characters present in the alignment. Several tools were used in the analysis of the
resulting files, including SAMtools 0.1.19<span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span>, BEDtools 2.17 <span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span> and BCFtools 0.2.0
(http://samtools.github.io/bcftools/bcftools.html)
<!--l. 34--><p class="indent" >
<a 
 id="x1-25r3"></a>
<!--l. 34--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.3.3</tspan>   <tspan font-family="cmbx" font-size="12">Taxonomic classification</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 36--><p class="indent" >       The mapped and unmapped reads were classified using Phylosift 1.0.1 <span class="cite">[<a 
href="#XDarling:2014ej">18</a>]</span>, using
the provided set of marker genes. This set included all of the January 2007 genomes
assembled from the Lake Tyrrell community <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>]</span>, but did not included the newest
one from the August 2007 community <span class="cite">[<a 
href="#XPodell:2013fp">66</a>]</span>.
<!--l. 38--><p class="indent" >
<a 
 id="x1-26r4"></a>
<!--l. 38--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.3.4</tspan>   <tspan font-family="cmbx" font-size="12">Variation analysis</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 40--><p class="indent" >       The mapped reads were processed using Picard Tools 1.99
(http://picard.sourceforge.net), and then GATK XXX (<span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span> was used to realign indel
regions to the reference genomes. The corrected files were processed with Freebayes
v9.9.2-29-g9ed353c <span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span> (ploidy:1, minimum base quality: 20, minimum mapping quality:
30). Quantification of the type of variations was done using SNPeff <span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span>, and custom
Python scripts.
<!--l. 42--><p class="indent" >       Calculation of the pN/pS values was done using custom Python scripts (available
in XXX), based on the approach used in Tai <tspan font-family="cmti" font-size="12">et al. </tspan><span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span>. For each gene the pN/pS value
was calculated as:
<div class="center" 
>
<!--l. 44--><p class="noindent" >
                                                                                <img 
src="template166x.png" alt="PICT" >
<img 
src="template167x.png" alt="PICT" >
                                                                                
<!--l. 49--><p class="noindent" > <img 
src="template164x.png" alt="pN    ObNseurmvebdenroofnnsyonnosnyynmoonuymsomuustasititeosns
pS-=  -Observed-synonymousmutations--
         Numberofsynonymoussites   "  class="math" >                     </div>
<!--l. 52--><p class="indent" >
<a 
 id="x1-27r5"></a>
<!--l. 52--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.3.5</tspan>   <tspan font-family="cmbx" font-size="12">Computational resources and data availability</tspan>
<a 
 id="Q1-1-8"></a>
<!--l. 54--><p class="indent" >       All of the analysis were carried out using a large cluster instance (c3.8xlarge: 32
Intel Xeon E5-2680 v2 cores, 60 Gb RAM) on the elastic cloud computing resource at
Amazon. Plotting and calculations were carried out using custom developed Python
scripts. The majority of the code and preliminary analyses are available on a Github
repository (XXX).
                                                                                <img 
src="template168x.png" alt="PICT" >
<img 
src="template169x.png" alt="PICT" >
                                                                                
<!--l. 56--><p class="indent" >       <hr class="float"><div class="float" 
>
                                                                                <img 
src="template170x.png" alt="PICT" >
<img 
src="template171x.png" alt="PICT" >
                                                                                
<a 
 id="x1-28r1"></a>
<tspan font-family="cmbx" font-size="12">Table</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.1</tspan>: List of the Lake Tyrrell habitat-specific genomes used for read mapping
<div class="center" 
>
<!--l. 58--><p class="noindent" >
<!--l. 59--><p class="noindent" > <table id="TBL-8" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-8-1g"><col 
id="TBL-8-1"><col 
id="TBL-8-2"><col 
id="TBL-8-3"><col 
id="TBL-8-4"><col 
id="TBL-8-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-8-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-1-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Genome name (abbrv)</tspan>   </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-1-2"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Length</tspan>     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-1-3"  
class="td11"> <tspan font-family="cmbx" font-size="12">G+C pct </tspan></td><td  style="white-space:wrap; text-align:left;" id="TBL-8-1-4"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmbx" font-size="12">N</tspan>
 <tspan font-family="cmbx" font-size="12">scaffolds</tspan>   </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-1-5"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmbx" font-size="12">Reference</tspan> </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-8-2-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-2-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Haloquadratum       walsbyi</tspan>
 J0HQW1                           </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-2-2"  
class="td11"> <!--l. 84--><p class="noindent" >3,549,539    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-2-3"  
class="td11"> 47            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-2-4"  
class="td11"> <!--l. 84--><p class="noindent" >1               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-2-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-3-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-3-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Haloquadratum       walsbyi</tspan>
 J0HQW2                           </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-3-2"  
class="td11"> <!--l. 84--><p class="noindent" >3,475,501    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-3-3"  
class="td11"> 49            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-3-4"  
class="td11"> <!--l. 84--><p class="noindent" >1               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-3-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-4-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-4-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Haloquadratum            </tspan>sp.
  J07HQX50                        </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-4-2"  
class="td11"> <!--l. 84--><p class="noindent" >3,019,909    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-4-3"  
class="td11"> 50            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-4-4"  
class="td11"> <!--l. 84--><p class="noindent" >2               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-4-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-5-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-5-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Nanosalinarum            </tspan>sp.
  J07AB56                           </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-5-2"  
class="td11"> <!--l. 84--><p class="noindent" >1,215,802    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-5-3"  
class="td11"> 56            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-5-4"  
class="td11"> <!--l. 84--><p class="noindent" >3               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-5-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-6-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-6-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Nanosalinarum            </tspan>sp.
  J07AB43                           </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-6-2"  
class="td11"> <!--l. 84--><p class="noindent" >1,277,157    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-6-3"  
class="td11"> 43            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-6-4"  
class="td11"> <!--l. 84--><p class="noindent" >7               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-6-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-7-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-7-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halonotius </tspan>sp. J07HN4        </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-7-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,888,659    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-7-3"  
class="td11"> 61            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-7-4"  
class="td11"> <!--l. 84--><p class="noindent" >2               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-7-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-8-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-8-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halonotius </tspan>sp. J07HN6        </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-8-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,529,000    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-8-3"  
class="td11"> 63            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-8-4"  
class="td11"> <!--l. 84--><p class="noindent" >6               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-8-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-9-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-9-1"  
class="td11"> <!--l. 84--><p class="noindent" >uncultured   archaeon   sp.
  J07HX64                           </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-9-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,982,938    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-9-3"  
class="td11"> 64            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-9-4"  
class="td11"> <!--l. 84--><p class="noindent" >1               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-9-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-10-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-10-1"  
class="td11"> <!--l. 84--><p class="noindent" >uncultured   archaeon   sp.
  J07HX5                            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-10-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,040,945    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-10-3"  
class="td11"> 60            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-10-4"  
class="td11"> <!--l. 84--><p class="noindent" >1               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-10-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-11-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-11-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halobaculum </tspan>sp. J07HB67    </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-11-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,649,547    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-11-3"  
class="td11"> 67            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-11-4"  
class="td11"> <!--l. 84--><p class="noindent" >3               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-11-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-12-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-12-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halorubrum </tspan>sp. J07HR59     </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-12-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,120,805    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-12-3"  
class="td11"> 59            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-12-4"  
class="td11"> <!--l. 84--><p class="noindent" >7               </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-12-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-13-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-13-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Salinibacter </tspan>sp. J07SB67      </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-13-2"  
class="td11"> <!--l. 84--><p class="noindent" >1,931,021    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-13-3"  
class="td11"> 67            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-13-4"  
class="td11"> <!--l. 84--><p class="noindent" >443            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-13-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-14-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-14-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halorubrum </tspan>sp. A07HR60     </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-14-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,876,249    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-14-3"  
class="td11"> 59            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-14-4"  
class="td11"> <!--l. 84--><p class="noindent" >14             </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-14-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013fp">66</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-15-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-15-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halonotius </tspan>sp. A07HN63      </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-15-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,392,686    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-15-3"  
class="td11"> 63            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-15-4"  
class="td11"> <!--l. 84--><p class="noindent" >37             </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-15-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013fp">66</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-16-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-16-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Halorubrum </tspan>sp. A07HR67     </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-16-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,890,468    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-16-3"  
class="td11"> 67            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-16-4"  
class="td11"> <!--l. 84--><p class="noindent" >16             </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-16-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013fp">66</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-17-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-17-1"  
class="td11"> <!--l. 84--><p class="noindent" >uncultured           archaeon
  A07HB70                          </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-17-2"  
class="td11"> <!--l. 84--><p class="noindent" >2,389,822    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-17-3"  
class="td11"> 71            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-17-4"  
class="td11"> <!--l. 84--><p class="noindent" >15             </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-17-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XPodell:2013fp">66</a>]</span>    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-18-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-18-1"  
class="td11"> <!--l. 84--><p class="noindent" ><tspan font-family="cmti" font-size="12">Candidatus      </tspan>Halobonum
  tyrellensis G22                   </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-18-2"  
class="td11"> <!--l. 84--><p class="noindent" >3,675,087    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-18-3"  
class="td11"> 70            </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-18-4"  
class="td11"> <!--l. 84--><p class="noindent" >72             </td><td  style="white-space:wrap; text-align:left;" id="TBL-8-18-5"  
class="td11"> <!--l. 84--><p class="noindent" ><span class="cite">[<a 
href="#XUgalde:2013hb">86</a>]</span>    </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-8-19-"><td  style="white-space:wrap; text-align:left;" id="TBL-8-19-1"  
class="td11"> <!--l. 84--><p class="noindent" >                         </td></tr></table>                                               </div>
                                                                                <img 
src="template172x.png" alt="PICT" >
<img 
src="template173x.png" alt="PICT" >
                                                                                
       </div><hr class="endfloat" />
                                                                                <img 
src="template174x.png" alt="PICT" >
<img 
src="template175x.png" alt="PICT" >
                                                                                
<a 
 id="x1-29r4"></a>
<!--l. 102--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.4</tspan>    <tspan font-family="cmbx" font-size="12">Results and Discussion</tspan>
<a 
 id="Q1-1-9"></a>
<!--l. 104--><p class="indent" >
<a 
 id="x1-30r1"></a>
<!--l. 104--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.4.1</tspan>   <tspan font-family="cmbx" font-size="12">Overview of the Illumina dataset for January and</tspan>
         <tspan font-family="cmbx" font-size="12">August of 2007</tspan>
<a 
 id="Q1-1-9"></a>
<!--l. 106--><p class="indent" >       Between 71-74% of the original reads were retained after trimming and quality
filtering (Table <tspan font-family="cmbx" font-size="12">??</tspan>). An approach to easily visualize the community composition and
identify broad differences in population composition between samples, is to quantify the
G+C content of the reads and plot the abundance by bins (usually of size 1%)
<span class="cite">[<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XGhai:2012fb">27</a>,&#x00A0;<a 
href="#XPodell:2013fp">66</a>]</span>. Figure <tspan font-family="cmbx" font-size="12">??</tspan>, shows the differences between the four libraries, and
highlights the previously assembled genomes from the Lake Tyrrell microbial
community <span class="cite">[<a 
href="#XNarasingarao:2012kp">54</a>,&#x00A0;<a 
href="#XPodell:2013kx">67</a>,&#x00A0;<a 
href="#XPodell:2013fp">66</a>]</span>. The plot allows to identify the different populations present
in the January versus the August libraries, in particular where the January
community is dominated by organisms with lower G+C content, compared to
the August community. This agrees with previous observations in the same
microbial community <span class="cite">[<a 
href="#XPodell:2013fp">66</a>]</span>, where the main driver of these differences was suggested
to be the ionic composition of the water column, in particular magnesium
(Table <tspan font-family="cmbx" font-size="12">??</tspan>). Microorganisms like <tspan font-family="cmti" font-size="12">Haloquadratum </tspan>(J07HQW1, J07HQW2 and
J07HQX50) are more dominant in the January samples, compared to the August
samples.
<!--l. 108--><p class="indent" >       Another interesting observation that emerge from this G+C plot, is that the
August samples have similar compositions based on the G+C content, while the
January samples show differences. This could be possibly attributed to the differences in
magnesium concentrations (Table <tspan font-family="cmbx" font-size="12">??</tspan>) between the two days in January, which could
explain the increase in the <tspan font-family="cmti" font-size="12">Haloquadratum </tspan>populations on the January 25th sample, and
in general of populations with a lower G+C content. Looking at possible explanations
for this, we found on the weather records, that a storm occurred previous to the January
                                                                                <img 
src="template176x.png" alt="PICT" >
<img 
src="template177x.png" alt="PICT" >
                                                                                
23th sampling, suggesting that the input of freshwater diluted the salt concentrations in
the water column, reducing the magnesium concentration. After two days,
due to water evaporation and other environmental factors, the magnesium
concentration raised, explaining the difference that we see between the two
days.
                                                                                <img 
src="template178x.png" alt="PICT" >
<img 
src="template179x.png" alt="PICT" >
                                                                                
<!--l. 113--><p class="indent" >       <hr class="float"><div class="float" 
>
                                                                                <img 
src="template180x.png" alt="PICT" >
<img 
src="template181x.png" alt="PICT" >
                                                                                
<a 
 id="x1-31r2"></a>
<tspan font-family="cmbx" font-size="12">Table</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.2</tspan>: Summary of the Illumina HiSeq libraries for each of the four samples.
 <table id="TBL-11" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-11-1g"><col 
id="TBL-11-1"><col 
id="TBL-11-2"><col 
id="TBL-11-3"><col 
id="TBL-11-4"><col 
id="TBL-11-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-1-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-1-1"  
class="td11"> <!--l. 123--><p class="noindent" >
 <tspan font-family="cmbx" font-size="12">Library</tspan>
 <tspan font-family="cmbx" font-size="12">name</tspan>        </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-1-2"  
class="td11">          <!--l. 123--><p class="noindent" >
     <tspan font-family="cmbx" font-size="12">Total</tspan>
       <tspan font-family="cmbx" font-size="12">reads</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-1-3"  
class="td11">               <!--l. 123--><p class="noindent" >
     <tspan font-family="cmbx" font-size="12">Read-pairs</tspan>
              <tspan font-family="cmbx" font-size="12">after</tspan>
                <tspan font-family="cmbx" font-size="12">QC</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-1-4"  
class="td11">                <!--l. 123--><p class="noindent" >
       <tspan font-family="cmbx" font-size="12">Unpaired</tspan>
               <tspan font-family="cmbx" font-size="12">reads</tspan>
               <tspan font-family="cmbx" font-size="12">after</tspan>
                 <tspan font-family="cmbx" font-size="12">QC</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-1-5"  
class="td11">          <!--l. 123--><p class="noindent" >
     <tspan font-family="cmbx" font-size="12">Total</tspan>
       <tspan font-family="cmbx" font-size="12">bases</tspan>
        <tspan font-family="cmbx" font-size="12">(Gb)</tspan> </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-2-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-2-1"  
class="td11"> <!--l. 123--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">January</tspan>
 <tspan font-family="cmti" font-size="12">23</tspan>              </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-2-2"  
class="td11">          <!--l. 123--><p class="noindent" >
  49,963,357  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-2-3"  
class="td11">               <!--l. 123--><p class="noindent" >
      37,016,243  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-2-4"  
class="td11">                <!--l. 123--><p class="noindent" >
        7,679,004  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-2-5"  
class="td11">          <!--l. 123--><p class="noindent" >
   7,978.18  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-3-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-3-1"  
class="td11"> <!--l. 123--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">January</tspan>
 <tspan font-family="cmti" font-size="12">25</tspan>              </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-3-2"  
class="td11">          <!--l. 123--><p class="noindent" >
  39,400,015  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-3-3"  
class="td11">               <!--l. 123--><p class="noindent" >
      29,444,267  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-3-4"  
class="td11">                <!--l. 123--><p class="noindent" >
        5,894,815  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-3-5"  
class="td11">          <!--l. 123--><p class="noindent" >
    6333.12  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-4-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-4-1"  
class="td11"> <!--l. 123--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">August</tspan>
 <tspan font-family="cmti" font-size="12">7</tspan>                </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-4-2"  
class="td11">          <!--l. 123--><p class="noindent" >
  46,472,319  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-4-3"  
class="td11">               <!--l. 123--><p class="noindent" >
      33,485,834  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-4-4"  
class="td11">                <!--l. 123--><p class="noindent" >
        7,659,231  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-4-5"  
class="td11">          <!--l. 123--><p class="noindent" >
    7266.38  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-5-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-5-1"  
class="td11"> <!--l. 123--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">August</tspan>
 <tspan font-family="cmti" font-size="12">9</tspan>                </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-5-2"  
class="td11">          <!--l. 123--><p class="noindent" >
  40,256,946  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-5-3"  
class="td11">               <!--l. 123--><p class="noindent" >
      28,843,346  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-5-4"  
class="td11">                <!--l. 123--><p class="noindent" >
        6,812,171  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-5-5"  
class="td11">          <!--l. 123--><p class="noindent" >
    6276.12  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-6-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-11-6-1"  
class="td11"> <!--l. 123--><p class="noindent" >            </td></tr></table>
                                                                                <img 
src="template182x.png" alt="PICT" >
<img 
src="template183x.png" alt="PICT" >
                                                                                
       </div><hr class="endfloat" />
<!--l. 129--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template184x.png" alt="PICT" >
<img 
src="template185x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template186x.png" alt="PICT" >
<img 
src="template188x.png" alt="PICT" >
                                                                                
<!--l. 131--><p class="noindent" ><img 
src="template187x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template187x.png" src="Chapter5/Figures/GC_content_HiSeqLibs.pdf"  
--> <a 
 id="x1-32r1"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.1</tspan>: GC plot of libraries
                                                                                <img 
src="template189x.png" alt="PICT" >
<img 
src="template190x.png" alt="PICT" >
                                                                                
<!--l. 134--><p class="indent" >       </div><hr class="endfigure">
                                                                                <img 
src="template191x.png" alt="PICT" >
<img 
src="template192x.png" alt="PICT" >
                                                                                
<!--l. 140--><p class="indent" >       <hr class="float"><div class="float" 
>
                                                                                <img 
src="template193x.png" alt="PICT" >
<img 
src="template194x.png" alt="PICT" >
                                                                                
<a 
 id="x1-33r3"></a>
<tspan font-family="cmbx" font-size="12">Table</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.3</tspan>: <tspan font-family="cmbx" font-size="12">Table</tspan> <tspan font-family="cmbx" font-size="12">??</tspan><tspan font-family="cmbx" font-size="12">: </tspan>Physical and chemical composition of the Lake Tyrrell
water samples. Concentrations are given in units of mmol L<sup class="textsuperscript"><tspan font-family="tcrm" font-size="1095">-</tspan><tspan font-family="cmr" font-size="10">1</tspan></sup>
                                                                                <img 
src="template195x.png" alt="PICT" >
<img 
src="template196x.png" alt="PICT" >
                                                                                
       </div><hr class="endfloat" />
                                                                                <img 
src="template197x.png" alt="PICT" >
<img 
src="template198x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template199x.png" alt="PICT" >
<img 
src="template200x.png" alt="PICT" >
                                                                                
<!--l. 147--><p class="indent" >       <hr class="float"><div class="float" 
>
                                                                                <img 
src="template201x.png" alt="PICT" >
<img 
src="template202x.png" alt="PICT" >
                                                                                
<div class="tabular"> <table id="TBL-12" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-12-1g"><col 
id="TBL-12-1"><col 
id="TBL-12-2"><col 
id="TBL-12-3"><col 
id="TBL-12-4"><col 
id="TBL-12-5"><col 
id="TBL-12-6"><col 
id="TBL-12-7"><col 
id="TBL-12-8"><col 
id="TBL-12-9"><col 
id="TBL-12-10"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-12-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-1"  
class="td11"> <tspan font-family="cmbx" font-size="12">Sample </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-2"  
class="td11"> <tspan font-family="cmbx" font-size="12">Temp</tspan> <tspan font-family="cmsy" font-size="8">&#x2218;</tspan>C  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-3"  
class="td11"> <tspan font-family="cmbx" font-size="12">Total ionic strength </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-4"  
class="td11">  <tspan font-family="cmbx" font-size="12">Na  </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-5"  
class="td11">  <tspan font-family="cmbx" font-size="12">K   </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-6"  
class="td11"> <tspan font-family="cmbx" font-size="12">Mg </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-7"  
class="td11"> <tspan font-family="cmbx" font-size="12">Ca </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-8"  
class="td11"> <tspan font-family="cmbx" font-size="12">Cl </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-9"  
class="td11">  <tspan font-family="cmbx" font-size="12">Cl   </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-10"  
class="td11"> <tspan font-family="cmbx" font-size="12">SO42 </tspan></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-1"  
class="td11">  <tspan font-family="cmti" font-size="12">Jan 23   </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-2"  
class="td11">   21.6     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-3"  
class="td11">         pH              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-4"  
class="td11"> 5,721  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-5"  
class="td11"> 4,338  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-6"  
class="td11">  32   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-7"  
class="td11"> 298  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-8"  
class="td11"> 10  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-9"  
class="td11"> 5,345  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-10"  
class="td11"> 123.6  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-1"  
class="td11">  <tspan font-family="cmti" font-size="12">Jan 25   </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-2"  
class="td11">   27.9     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-3"  
class="td11">         7.09              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-4"  
class="td11"> 5,950  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-5"  
class="td11"> 4,163  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-6"  
class="td11">  43   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-7"  
class="td11"> 419  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-8"  
class="td11"> 11  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-9"  
class="td11"> 5,291  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-10"  
class="td11"> 170.5  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-1"  
class="td11">  <tspan font-family="cmti" font-size="12">Aug 6    </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-2"  
class="td11">    9.9      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-3"  
class="td11">         7.00              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-4"  
class="td11"> 4,403  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-5"  
class="td11"> 3,724  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-6"  
class="td11">  19   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-7"  
class="td11"> 126  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-8"  
class="td11"> 15  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-9"  
class="td11"> 4,298  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-10"  
class="td11">   50    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-1"  
class="td11">  <tspan font-family="cmti" font-size="12">Aug 8    </tspan></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-2"  
class="td11">   11.5     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-3"  
class="td11">         7.01              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-4"  
class="td11"> 4,060  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-5"  
class="td11"> 3,557  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-6"  
class="td11">  18   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-7"  
class="td11"> 117  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-8"  
class="td11"> 14  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-9"  
class="td11"> 3,830  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-10"  
class="td11">   47    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-1"  
class="td11">         </td></tr></table></div>
                                                                                <img 
src="template203x.png" alt="PICT" >
<img 
src="template204x.png" alt="PICT" >
                                                                                
       </div><hr class="endfloat" />
                                                                                <img 
src="template205x.png" alt="PICT" >
<img 
src="template206x.png" alt="PICT" >
                                                                                
<a 
 id="x1-34r2"></a>
<!--l. 168--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.4.2</tspan>   <tspan font-family="cmbx" font-size="12">Genome abundance and community structure, based on</tspan>
         <tspan font-family="cmbx" font-size="12">read mapping</tspan>
<a 
 id="Q1-1-12"></a>
<!--l. 170--><p class="indent" >       All of the reads that passed the quality filters were mapped against the set of
reference genomes (Table 5.1). The summary of this mapping (Table <tspan font-family="cmbx" font-size="12">??</tspan>), shows the
difference in the number of reads that mapped to each genome. In both January
samples, the <tspan font-family="cmti" font-size="12">Haloquadratum </tspan>J07HWQ1 and J07HWQ2 genomes, recruited the largest
number of reads, which agrees with previous estimation of their abundance in the Lake
Tyrrell community <span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>. It is important to highlight that number of mapped reads to
each genome, do not reflect the real abundance of each organism in the community.
Given the strict criteria used for mapping, it is likely that we are missing information on
some of these genomes (more on this later), and also several of these genomes
represent closely related species, sharing the same DNA sequences. In addition,
variations due to genome size and even genome copy number need to be taken in
consideration. This explains, for example, why the <tspan font-family="cmti" font-size="12">Nanohaloarchaea </tspan>genomes (J07AB56
and J07AB43) do not recruit a large percentage of the reads, in contrast with
their estimated high abundance in this community <span class="cite">[<a 
href="#XPodell:2013kx">67</a>]</span>. This stringent criteria,
also explain why the genome of <tspan font-family="cmti" font-size="12">Candidatus </tspan>Halobonum tyrrellensis (G22),
recruits a low number of reads from the dataset. This particular organisms, is
an interesting situation, as it was isolated from surface waters collected in
August of 2007, and by 16S showed to be similar to the <tspan font-family="cmti" font-size="12">Halobaculum </tspan>genomes
assembled from the January and August samples (J07HB67 and A07HB70).
Further analysis using the complete sequence, showed that it was a different
population, and possibly a new Genus within the Haloarchaea <span class="cite">[<a 
href="#XUgalde:2013hb">86</a>]</span>. This is a
clear example on how in many situations, isolates from an environment are not
representative of the most abundant members that can be found living in that
environment.
<!--l. 172--><p class="indent" >       A visualization of the numbers of mapped reads at different identity levels
(Figure <tspan font-family="cmbx" font-size="12">??</tspan>), shows that for all the genomes (with the exception of G22), the majority of
the reads mapped at a 100% identity, and in none case was lower than 85%. This
                                                                                <img 
src="template207x.png" alt="PICT" >
<img 
src="template208x.png" alt="PICT" >
                                                                                
reflects the stringency of the parameters used for mapping, and also allows
to be confident that the reads are derived from that particular population.
Looking more in detail to each of the individual genomes, these plots already
show some information on the possible strain diversity present in each of the
populations. In the case of the <tspan font-family="cmti" font-size="12">Haloquadratum </tspan>genomes, most of the reads
mapped at identities of 95% or higher, reflecting a low level of population
heterogeneity among these populations. On the other extreme, in the case of the
<tspan font-family="cmti" font-size="12">Salinibacter </tspan>populations, we can observe a range of identities going from 88% and
higher.
<!--l. 174--><p class="indent" >       Given the deep coverage of the community in this four libraries (an average of
4.3 billion nucleotides in each one), this dataset represent not only the most
abundant members of the community, represented in the reference genomes used
for mapping, but also allows access to less abundant organisms. To quickly
evaluate the differences between the mapped and unmapped reads, we used a
taxonomic classification approach to estimate difference between the mapped and
unmapped reads on each dataset. For example, for the mapped reads in the January
23 dataset (Figure <tspan font-family="cmbx" font-size="12">??</tspan>), as expected the majority of the mapped reads were
classified as <tspan font-family="cmti" font-size="12">Haloquadratum</tspan>, followed by the <tspan font-family="cmti" font-size="12">Nanohaloarchaea</tspan>. In comparison,
the unmapped reads for this library (Figure <tspan font-family="cmbx" font-size="12">??</tspan>), shows a diversity of groups,
including a high percentage of Bacterial reads. These difference in taxonomic
composition between the set of mapped and unmapped reads, can be visualize in a
two-dimensional plot using an Edge principal component (EPCA) analysis <span class="cite">[<tspan font-family="cmbx" font-size="12">?</tspan>]</span>
derived from the Phylosift results. Plotting the first two components shows that
based on the predicted taxonomic composition of the community, the reads
separate between the mapped and unmapped groups, and in addition there is a
separation by season in the case of the unmapped reads. For the mapped reads, we
observe that the group separately from the unmapped, but due to technical
problems, the data available is only for two of the libraries, one from each
month.
<!--l. 176--><p class="indent" >       The taxonomic analysis suggests the presence of novel groups (not represented in
the current set of genomes used for mapping) present in the community, and that with
                                                                                <img 
src="template209x.png" alt="PICT" >
<img 
src="template210x.png" alt="PICT" >
                                                                                
further analysis could be recovered. In the present work, I will focus only on the
genomes and its mapped reads, as this genomes provide an already validated set of
habitat-specific genomes that can be used for the analysis of diversity in this microbial
community.
                                                                                <img 
src="template211x.png" alt="PICT" >
<img 
src="template212x.png" alt="PICT" >
                                                                                
<!--l. 178--><p class="indent" >       <hr class="float"><div class="float" 
>
                                                                                <img 
src="template213x.png" alt="PICT" >
<img 
src="template214x.png" alt="PICT" >
                                                                                
<a 
 id="x1-35r3"></a>
<tspan font-family="cmbx" font-size="12">Table</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.3</tspan>:  Number  of  reads  from  each  library,  recruited  to  the  Lake  Tyrrell
reference genomes
                                                                                <img 
src="template215x.png" alt="PICT" >
<img 
src="template216x.png" alt="PICT" >
                                                                                
 <table id="TBL-15" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-15-1g"><col 
id="TBL-15-1"><col 
id="TBL-15-2"><col 
id="TBL-15-3"><col 
id="TBL-15-4"><col 
id="TBL-15-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-15-1-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-1-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmbx" font-size="12">Genome</tspan>    </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-1-2"  
class="td11">          <!--l. 202--><p class="noindent" >
       <tspan font-family="cmbx" font-size="12">Jan</tspan>
           <tspan font-family="cmbx" font-size="12">23</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-1-3"  
class="td11">               <!--l. 202--><p class="noindent" >
           <tspan font-family="cmbx" font-size="12">Jan</tspan>
                 <tspan font-family="cmbx" font-size="12">25</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-1-4"  
class="td11">                <!--l. 202--><p class="noindent" >
            <tspan font-family="cmbx" font-size="12">Aug</tspan>
                   <tspan font-family="cmbx" font-size="12">07</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-1-5"  
class="td11">          <!--l. 202--><p class="noindent" >
      <tspan font-family="cmbx" font-size="12">Aug</tspan>
           <tspan font-family="cmbx" font-size="12">09</tspan> </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-15-2-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-2-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HWQ1</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-2-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  9,712,976  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-2-3"  
class="td11">               <!--l. 202--><p class="noindent" >
      10,347,084  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-2-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        1,802,421  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-2-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,385,337  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-3-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-3-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HWQ2</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-3-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  7,311,175  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-3-3"  
class="td11">               <!--l. 202--><p class="noindent" >
       9,428,490  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-3-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        1,628,137  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-3-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,301,141  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-4-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-4-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HQX50</tspan>    </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-4-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    922,138  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-4-3"  
class="td11">               <!--l. 202--><p class="noindent" >
       1,041,326  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-4-4"  
class="td11">                <!--l. 202--><p class="noindent" >
         501,477  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-4-5"  
class="td11">          <!--l. 202--><p class="noindent" >
    330,307  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-5-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-5-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07AB56</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-5-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    565,197  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-5-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         266,831  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-5-4"  
class="td11">                <!--l. 202--><p class="noindent" >
         194,445  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-5-5"  
class="td11">          <!--l. 202--><p class="noindent" >
    165,278  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-6-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-6-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07AB43</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-6-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    760,203  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-6-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         486,360  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-6-4"  
class="td11">                <!--l. 202--><p class="noindent" >
          63,209  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-6-5"  
class="td11">          <!--l. 202--><p class="noindent" >
     64295  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-7-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-7-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HN4</tspan>       </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-7-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  2,149,204  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-7-3"  
class="td11">               <!--l. 202--><p class="noindent" >
       2,249,692  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-7-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        1,306,287  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-7-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,089,673  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-8-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-8-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HN6</tspan>       </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-8-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    592,818  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-8-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         831,367  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-8-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        1,027,472  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-8-5"  
class="td11">          <!--l. 202--><p class="noindent" >
    911,341  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-9-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-9-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HX64</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-9-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  4,167,113  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-9-3"  
class="td11">               <!--l. 202--><p class="noindent" >
       1,819,023  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-9-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        1,202,103  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-9-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,144,206  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-10-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-10-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HX5</tspan>       </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-10-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  2,106,559  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-10-3"  
class="td11">               <!--l. 202--><p class="noindent" >
       1,382,371  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-10-4"  
class="td11">                <!--l. 202--><p class="noindent" >
         673,972  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-10-5"  
class="td11">          <!--l. 202--><p class="noindent" >
    539,843  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-11-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-11-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HB67</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-11-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,124,816  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-11-3"  
class="td11">               <!--l. 202--><p class="noindent" >
       1,128,191  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-11-4"  
class="td11">                <!--l. 202--><p class="noindent" >
         125,973  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-11-5"  
class="td11">          <!--l. 202--><p class="noindent" >
     84,643  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-12-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-12-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HR59</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-12-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    839,856  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-12-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         310,496  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-12-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        2,105,598  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-12-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,693,772  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-13-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-13-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HB70</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-13-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    550,429  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-13-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         277,030  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-13-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        1,970,106  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-13-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,866,967  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-14-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-14-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HR67</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-14-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    563,043  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-14-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         270,602  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-14-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        2,166,129  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-14-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,680,150  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-15-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-15-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HN63</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-15-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    547,808  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-15-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         786,856  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-15-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        1,126,032  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-15-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  1,003,322  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-16-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-16-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HR60</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-16-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  2,126,700  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-16-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         758,549  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-16-4"  
class="td11">                <!--l. 202--><p class="noindent" >
        5,405,933  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-16-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  4,362,857  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-17-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-17-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">G22</tspan>            </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-17-2"  
class="td11">          <!--l. 202--><p class="noindent" >
     62,983  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-17-3"  
class="td11">               <!--l. 202--><p class="noindent" >
          39,696  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-17-4"  
class="td11">                <!--l. 202--><p class="noindent" >
          72,261  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-17-5"  
class="td11">          <!--l. 202--><p class="noindent" >
     66,778  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-18-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-18-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07SB</tspan>         </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-18-2"  
class="td11">          <!--l. 202--><p class="noindent" >
    797,957  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-18-3"  
class="td11">               <!--l. 202--><p class="noindent" >
         211,306  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-18-4"  
class="td11">                <!--l. 202--><p class="noindent" >
         737,471  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-18-5"  
class="td11">          <!--l. 202--><p class="noindent" >
    673,630  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-19-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-19-1"  
class="td11"> <!--l. 202--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">Unmapped</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-19-2"  
class="td11">          <!--l. 202--><p class="noindent" >
  45,344,829  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-19-3"  
class="td11">               <!--l. 202--><p class="noindent" >
      31,913,858  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-19-4"  
class="td11">                <!--l. 202--><p class="noindent" >
       51,012,461  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-19-5"  
class="td11">          <!--l. 202--><p class="noindent" >
  44,849,506  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-20-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-15-20-1"  
class="td11"> <!--l. 202--><p class="noindent" >            </td></tr></table>
                                                                                <img 
src="template217x.png" alt="PICT" >
<img 
src="template218x.png" alt="PICT" >
                                                                                
       </div><hr class="endfloat" />
<!--l. 209--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template219x.png" alt="PICT" >
<img 
src="template220x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template221x.png" alt="PICT" >
<img 
src="template223x.png" alt="PICT" >
                                                                                
<!--l. 211--><p class="noindent" ><img 
src="template222x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template222x.png" src="/Users/juan/Dropbox/GeneticVariationLT/Figures/GenomeIdentityPlots.pdf"  
--> <a 
 id="x1-36r2"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.2</tspan>: Total number of recruited reads, grouped by identity. The X axis
shows the identity of the read to the reference genome (%), while the Y axis shows
the number of reads recruited at that identity (thousands of reads).
                                                                                <img 
src="template224x.png" alt="PICT" >
<img 
src="template225x.png" alt="PICT" >
                                                                                
<!--l. 214--><p class="indent" >       </div><hr class="endfigure">
<!--l. 218--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template226x.png" alt="PICT" >
<img 
src="template227x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template228x.png" alt="PICT" >
<img 
src="template231x.png" alt="PICT" >
                                                                                
<a 
 id="x1-37r1"></a>
<!--l. 223--><p class="noindent" > <img 
src="template229x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template229x.png" src="Chapter5/Figures/Jan23_Mapped.pdf"  
-->
<tspan font-family="cmr" font-size="10">(a) Mapped reads</tspan>
                                                                                <img 
src="template232x.png" alt="PICT" >
<img 
src="template233x.png" alt="PICT" >
                                                                                
<a 
 id="x1-38r2"></a> <img 
src="template230x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template230x.png" src="Chapter5/Figures/Jan23_Unmapped.pdf"  
-->
<tspan font-family="cmr" font-size="10">(b) Unmapped reads</tspan>                                              <a 
 id="x1-39r3"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.3</tspan>: Taxonomic classification of the mapped and unmapped reads using
Phylosift <span class="cite">[<a 
href="#XDarling:2014ej">18</a>]</span>
                                                                                <img 
src="template234x.png" alt="PICT" >
<img 
src="template235x.png" alt="PICT" >
                                                                                
<!--l. 231--><p class="indent" >       </div><hr class="endfigure">
<!--l. 234--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template236x.png" alt="PICT" >
<img 
src="template237x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template238x.png" alt="PICT" >
<img 
src="template240x.png" alt="PICT" >
                                                                                
<!--l. 236--><p class="noindent" ><img 
src="template239x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template239x.png" src="Chapter5/Figures/Unmapped_Mapped_EPCA.pdf"  
--> <a 
 id="x1-40r4"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.4</tspan>: EPCA phylosift results
                                                                                <img 
src="template241x.png" alt="PICT" >
<img 
src="template242x.png" alt="PICT" >
                                                                                
<!--l. 239--><p class="indent" >       </div><hr class="endfigure">
                                                                                <img 
src="template243x.png" alt="PICT" >
<img 
src="template244x.png" alt="PICT" >
                                                                                
<a 
 id="x1-41r3"></a>
<!--l. 244--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.4.3</tspan>   <tspan font-family="cmbx" font-size="12">Genome coverage</tspan>
<a 
 id="Q1-1-16"></a>
<!--l. 246--><p class="indent" >       The numbers of reads mapped to each genome (Table <tspan font-family="cmbx" font-size="12">??</tspan>), suggests differences in
the relative abundance of certain populations between seasons. For example, the
<tspan font-family="cmti" font-size="12">Haloquadratum </tspan>genomes (J07HWQ1 and J07HWQ2), recruited more reads from the
January datasets than from the August ones. In contrast, genomes that were assembled
from August samples, like A07HR60, recruited more reads from August. However, these
numbers do not provide a complete picture of the coverage, because it is possible than
certain areas of the genome have a higher coverage than other ones. Also, some
regions could show a differential coverage, with more reads mapping from one
season.
<!--l. 251--><p class="indent" >       To evaluate the coverage of these reads across the genomes, we looked
across
<!--l. 255--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template245x.png" alt="PICT" >
<img 
src="template246x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template247x.png" alt="PICT" >
<img 
src="template249x.png" alt="PICT" >
                                                                                
<!--l. 257--><p class="noindent" ><img 
src="template248x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template248x.png" src="Chapter5/Figures/GeneDifferencesSeason.pdf"  
--> <a 
 id="x1-42r5"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.5</tspan>: Gene significant by seasons
                                                                                <img 
src="template250x.png" alt="PICT" >
<img 
src="template251x.png" alt="PICT" >
                                                                                
<!--l. 260--><p class="indent" >       </div><hr class="endfigure">
<a 
 id="x1-43r4"></a>
<!--l. 264--><p class="nopar" ><tspan font-family="cmbx" font-size="12">5.4.4</tspan>   <tspan font-family="cmbx" font-size="12">Variation and positive selection analysis</tspan>
<a 
 id="Q1-1-17"></a>
<!--l. 266--><p class="indent" >       Results of the snps (<tspan font-family="cmbx" font-size="12">??</tspan>). Low number of SNPs for G22, so I&#8217;ll leave out of the
analysis. Reasons? Comments about the trends?
<!--l. 269--><p class="indent" >       Compariosn of the rate of SNPs (independent of the type). DONE (Table)
<!--l. 271--><p class="indent" >       Venn diagram of the SNPs, can we collapse January samples and August
samples. DONE
<!--l. 274--><p class="indent" >       Genes with most of the SNPs Genes with no SNPs Use BEDTOOLS
ANNOTATE FOR THIS
<!--l. 280--><p class="indent" >       Dn/DS network diagram of DN/ds and selection, january versus august
                                                                                <img 
src="template252x.png" alt="PICT" >
<img 
src="template253x.png" alt="PICT" >
                                                                                
<!--l. 284--><p class="indent" >       <hr class="float"><div class="float" 
>
                                                                                <img 
src="template254x.png" alt="PICT" >
<img 
src="template255x.png" alt="PICT" >
                                                                                
<a 
 id="x1-44r4"></a>
<tspan font-family="cmbx" font-size="12">Table</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.4</tspan>: Change rate for SNPs
 <table id="TBL-18" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-18-1g"><col 
id="TBL-18-1"><col 
id="TBL-18-2"><col 
id="TBL-18-3"><col 
id="TBL-18-4"><col 
id="TBL-18-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-18-1-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-1-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmbx" font-size="12">Genome</tspan>    </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-1-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       <tspan font-family="cmbx" font-size="12">Jan</tspan>
           <tspan font-family="cmbx" font-size="12">23</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-1-3"  
class="td11">               <!--l. 307--><p class="noindent" >
           <tspan font-family="cmbx" font-size="12">Jan</tspan>
                 <tspan font-family="cmbx" font-size="12">25</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-1-4"  
class="td11">                <!--l. 307--><p class="noindent" >
            <tspan font-family="cmbx" font-size="12">Aug</tspan>
                   <tspan font-family="cmbx" font-size="12">07</tspan> </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-1-5"  
class="td11">          <!--l. 307--><p class="noindent" >
      <tspan font-family="cmbx" font-size="12">Aug</tspan>
           <tspan font-family="cmbx" font-size="12">09</tspan> </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-18-2-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-2-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HWQ1</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-2-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       261  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-2-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            262  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-2-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             267  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-2-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       270  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-3-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-3-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HWQ2</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-3-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       398  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-3-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            397  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-3-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             305  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-3-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       407  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-4-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-4-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HQX50</tspan>    </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-4-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       695  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-4-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            693  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-4-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             706  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-4-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       718  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-5-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-5-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07AB56</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-5-2"  
class="td11">          <!--l. 307--><p class="noindent" >
        99  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-5-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            103  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-5-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             202  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-5-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       192  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-6-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-6-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07AB43</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-6-2"  
class="td11">          <!--l. 307--><p class="noindent" >
        93  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-6-3"  
class="td11">               <!--l. 307--><p class="noindent" >
             94  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-6-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             137  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-6-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       139  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-7-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-7-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HN4</tspan>       </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-7-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       110  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-7-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            110  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-7-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             111  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-7-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       111  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-8-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-8-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HN6</tspan>       </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-8-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       348  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-8-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            383  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-8-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             351  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-8-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       356  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-9-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-9-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HX64</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-9-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       105  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-9-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            106  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-9-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             107  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-9-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       108  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-10-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-10-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HX5</tspan>       </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-10-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       108  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-10-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            109  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-10-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             110  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-10-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       111  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-11-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-11-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HB67</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-11-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       110  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-11-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            109  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-11-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             155  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-11-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       191  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-12-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-12-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07HR59</tspan>      </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-12-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       738  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-12-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            872  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-12-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             651  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-12-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       663  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-13-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-13-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HB70</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-13-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       139  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-13-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            142  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-13-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             137  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-13-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       137  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-14-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-14-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HR67</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-14-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       171  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-14-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            175  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-14-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             169  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-14-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       169  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-15-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-15-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HN63</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-15-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       581  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-15-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            504  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-15-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             471  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-15-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       476  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-16-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-16-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">A07HR60</tspan>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-16-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       264  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-16-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            271  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-16-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             256  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-16-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       257  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-17-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-17-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">G22</tspan>            </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-17-2"  
class="td11">          <!--l. 307--><p class="noindent" >
     24,126  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-17-3"  
class="td11">               <!--l. 307--><p class="noindent" >
          33,241  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-17-4"  
class="td11">                <!--l. 307--><p class="noindent" >
          29,898  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-17-5"  
class="td11">          <!--l. 307--><p class="noindent" >
     31,227  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-18-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-18-1"  
class="td11"> <!--l. 307--><p class="noindent" >
 <tspan font-family="cmti" font-size="12">J07SB</tspan>         </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-18-2"  
class="td11">          <!--l. 307--><p class="noindent" >
       165  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-18-3"  
class="td11">               <!--l. 307--><p class="noindent" >
            177  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-18-4"  
class="td11">                <!--l. 307--><p class="noindent" >
             164  </td><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-18-5"  
class="td11">          <!--l. 307--><p class="noindent" >
       164  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-19-"><td  style="white-space:nowrap; text-align:left; vertical-align:middle;" id="TBL-18-19-1"  
class="td11"> <!--l. 307--><p class="noindent" >            </td></tr></table>
                                                                                <img 
src="template256x.png" alt="PICT" >
<img 
src="template257x.png" alt="PICT" >
                                                                                
       </div><hr class="endfloat" />
<!--l. 312--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template258x.png" alt="PICT" >
<img 
src="template259x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template260x.png" alt="PICT" >
<img 
src="template262x.png" alt="PICT" >
                                                                                
<!--l. 314--><p class="noindent" ><img 
src="template261x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template261x.png" src="Chapter5/Figures/SNPsummary.pdf"  
--> <a 
 id="x1-45r6"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.6</tspan>: SNPsummary
                                                                                <img 
src="template263x.png" alt="PICT" >
<img 
src="template264x.png" alt="PICT" >
                                                                                
<!--l. 317--><p class="indent" >       </div><hr class="endfigure">
<!--l. 320--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template265x.png" alt="PICT" >
<img 
src="template266x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template267x.png" alt="PICT" >
<img 
src="template269x.png" alt="PICT" >
                                                                                
<!--l. 322--><p class="noindent" ><img 
src="template268x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template268x.png" src="Chapter5/Figures/Venn_JanuarySNPs.pdf"  
--> <a 
 id="x1-46r7"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.7</tspan>: JanuarySNPs
                                                                                <img 
src="template270x.png" alt="PICT" >
<img 
src="template271x.png" alt="PICT" >
                                                                                
<!--l. 325--><p class="indent" >       </div><hr class="endfigure">
<!--l. 328--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template272x.png" alt="PICT" >
<img 
src="template273x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template274x.png" alt="PICT" >
<img 
src="template276x.png" alt="PICT" >
                                                                                
<!--l. 330--><p class="noindent" ><img 
src="template275x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template275x.png" src="Chapter5/Figures/Venn_AugustSNPs.pdf"  
--> <a 
 id="x1-47r8"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.8</tspan>: AugustSNPs
                                                                                <img 
src="template277x.png" alt="PICT" >
<img 
src="template278x.png" alt="PICT" >
                                                                                
<!--l. 333--><p class="indent" >       </div><hr class="endfigure">
<!--l. 336--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template279x.png" alt="PICT" >
<img 
src="template280x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template281x.png" alt="PICT" >
<img 
src="template283x.png" alt="PICT" >
                                                                                
<!--l. 338--><p class="noindent" ><img 
src="template282x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template282x.png" src="Chapter5/Figures/Venn_JanAugSNPs.pdf"  
--> <a 
 id="x1-48r9"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.9</tspan>: BothSNPs
                                                                                <img 
src="template284x.png" alt="PICT" >
<img 
src="template285x.png" alt="PICT" >
                                                                                
<!--l. 341--><p class="indent" >       </div><hr class="endfigure">
<!--l. 344--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template286x.png" alt="PICT" >
<img 
src="template287x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template288x.png" alt="PICT" >
<img 
src="template290x.png" alt="PICT" >
                                                                                
<!--l. 346--><p class="noindent" ><img 
src="template289x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="template289x.png" src="Chapter5/Figures/SNP_freq_boxplot.pdf"  
--> <a 
 id="x1-49r10"></a>
<tspan font-family="cmbx" font-size="12">Figure</tspan><tspan font-family="cmbx" font-size="12">&#x00A0;5.10</tspan>: SNpp frequency
                                                                                <img 
src="template291x.png" alt="PICT" >
<img 
src="template292x.png" alt="PICT" >
                                                                                
<!--l. 349--><p class="indent" >       </div><hr class="endfigure">
<!--l. 352--><p class="indent" >       <hr class="figure"><div class="figure" 
>
                                                                                <img 
src="template293x.png" alt="PICT" >
<img 
src="template294x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template295x.png" alt="PICT" >
<img 
src="template297x.png" alt="PICT" >
                                                                                
                                                                                <img 
src="template298x.png" alt="PICT" >
<img 
src="template299x.png" alt="PICT" >
                                                                                
<!--l. 354--><p class="noindent" ><img 
src="template296x.png" alt="PIC" >
<img 
src="template301x.png" alt="PICT" >
                                                                                
