%
%
% UCSD Doctoral Dissertation Template
% -----------------------------------
% http://ucsd-thesis.googlecode.com
%
%


%% REQUIRED FIELDS -- Replace with the values appropriate to you

% No symbols, formulas, superscripts, or Greek letters are allowed
% in your title.
\title{Metagenomic Characterization of a Hypersaline Microbial Ecosystem}

\author{Juan A. Ugalde}
\degreeyear{2014}

% Master's Degree theses will NOT be formatted properly with this file.
\degreetitle{Doctor of Philosophy} 

\field{Marine Biology}
\chair{Professor Eric E. Allen}
% Uncomment the next line iff you have a Co-Chair
% \cochair{Professor Cochair Semimaster} 
%
% Or, uncomment the next line iff you have two equal Co-Chairs.
%\cochairs{Professor Chair Masterish}{Professor Chair Masterish}

%  The rest of the committee members  must be alphabetized by last name.
\othermembers{
Professor Farooq Azam\\ 
Professor Douglas H. Bartlett\\
Professor. Philip Bourne\\
Professor David T. Pride\\
Professor Forest Rohwer\\
}
\numberofmembers{6} % |chair| + |cochair| + |othermembers|


%% START THE FRONTMATTER
%
\begin{frontmatter}

%% TITLE PAGES
%
%  This command generates the title, copyright, and signature pages.
%
\makefrontmatter 

%% DEDICATION
%
%  You have three choices here:
%    1. Use the ``dedication'' environment. 
%       Put in the text you want, and everything will be formated for 
%       you. You'll get a perfectly respectable dedication page.
%   
%
%    2. Use the ``mydedication'' environment.  If you don't like the
%       formatting of option 1, use this environment and format things
%       however you wish.
%
%    3. If you don't want a dedication, it's not required.
%
%
\begin{dedication} 
  A mi familia, sin su apoyo nada de esto hubiera sido posible.    
\end{dedication}


% \begin{mydedication} % You are responsible for formatting here.
%   \vspace{1in}
%   \begin{flushleft}
% 	To me.
%   \end{flushleft}
%   
%   \vspace{2in}
%   \begin{center}
% 	And you.
%   \end{center}
% 
%   \vspace{2in}
%   \begin{flushright}
% 	Which equals us.
%   \end{flushright}
% \end{mydedication}



%% EPIGRAPH
%
%  The same choices that applied to the dedication apply here.
%
\begin{epigraph} % The style file will position the text for you.
  \emph{A beginning is the time for taking the most delicate care that the balances are correct.}\\
  ---Frank Herbert, \textit{Dune}.

  \emph{This is not the place to go into the specifies of which microbial genomes would be most useful. I would suggest, however, that a phylogenetic tree hang on the wall of every laboratory in which microbial genomes are being sequenced — for inspiration.}\\
  ---Carl Woese, \textit{A manifesto for microbial genomics}.
  

\end{epigraph}

% \begin{myepigraph} % You position the text yourself.
%   \vfil
%   \begin{center}
%     {\bf Think! It ain't illegal yet.}
% 
% 	\emph{---George Clinton}
%   \end{center}
% \end{myepigraph}


%% SETUP THE TABLE OF CONTENTS
%
\tableofcontents
\listoffigures  % Uncomment if you have any figures
\listoftables   % Uncomment if you have any tables



%% ACKNOWLEDGEMENTS
%
%  While technically optional, you probably have someone to thank.
%  Also, a paragraph acknowledging all coauthors and publishers (if
%  you have any) is required in the acknowledgements page and as the
%  last paragraph of text at the end of each respective chapter. See
%  the OGS Formatting Manual for more information.
%
\begin{acknowledgements} 

This work could not have been posible without the support of numerous persons over the last few years. I feel very fortunate for the opportunity that I had to met such amazing people over my years in San Diego.

First, I have to thank Eric Allen, for giving me the opportunity to join his research group. He provided me with enough guidance, but at the same time, with enough freedom to pursue multiple projects and ideas. His enthusiasm provided inspiration for this work, and for all my future projects. I also want to thank all my committee members, Farooq Azam, Douglas Bartlett, Philip Bourn, David Pride, and Forest Rohwer, for their comments, feedback and support through my scientific endeavors.

Multiple funding sources helped me along the years. In particular I want to acknowledge the financial support provided by a Fulbright-Conicyt fellowship during my first four years of graduate school.

I also have to thank all of the current members of the Allen Lab. In particular Sheila Podell, for all the help both with bioinformatics and scientific questions, and for being an amazing officemate. Also I have to thank Christine Shulse, for being such a great guide and for her friendship during my first years of grad school, and Jessica Blanton, for all the support and friendship during my last year of graduate school.

I am very grateful of the support of my friends outside the SIO community. In particular, I have to thank Jorge and Paula, for their constant support and friendship, from the first day that I arrived in San Diego.

Of course, nothing of this could have been possible without the support of my family. Thanks for always supporting me and believe in me. And of course, most of all, I want to thank Andrea, for her constant support, company, and patience. Without you, none of this could have been possible. Thank you.

\newpage

Chapter 2 is a full reprint of: De novo metagenomic assembly reveals abundant novel major lineage of Archaea in hypersaline communnities. P. Narasingarao, S. Podell, J.A. Ugalde, C. Brochier-Armanet, J.B. Emerson, J.J. Brocks, K.B. Heidelberg, J.F. Banfield and E.E. Allen. \textit{ISME Journal}, \textbf{6},81-93. 2012 (doi: 10.1038/ismej.2011.78), with permission from all coauthors.

Chapter 3 is a full reprint of: Xenorhodopsins, an enigmatic new class of microbial rhodopsins horizontally transferred between archaea and bacteria. J.A. Ugalde, S. Podell, P. Narasingarao and E.E. Allen. \textit{Biology Direct}, \textbf{6},52. 2011 (doi: 10.1186/1745-6150-6-52), with permission from all coauthors.

Chapter 4 is a full reprint of: Assembly-driven community genomics of a hypersaline microbial ecosystem. S. Podell, J.A. Ugalde, P. Narasingarao, J.F. Banfield, K.B. Heidelberg and E.E. Allen. \textit{PLoS One}, \textbf{8}4:e61692. 2013 (doi: 10.1371/journal.pone.0061692), with permission from all coauthors.

Appendix \ref{G22Genome} is a full reprint of: Draft Genome Sequence of "\textit{Candidatus} Halobonum tyrrellensis" Strain G22, Isolated from the Hypersaline Waters of Lake Tyrrell, Australia. J.A. Ugalde, P. Narasingarao, S. Kuo, S. Podell and E.E. Allen. \textit{Genome Announcements}, \textbf{1}(6):e01001-13. 2013. With permission from all coauthors.

\end{acknowledgements}


%% VITA
%
%  A brief vita is required in a doctoral thesis. See the OGS
%  Formatting Manual for more information.
%
\begin{vitapage}
\begin{vita}
  \item[2001-2003] Research assistant, Laboratory of Bioinformatics and Gene Expression, Universidad de Chile, Santiago, Chile.
  \item[2003-2004] Research assistant, Whitney Laboratory for Marine Biosciences, University of Florida.
  \item[2006] \emph{Licenciatura} in Molecular Biotechnology Engineering. Universidad de Chile, Santiago, Chile.
  \item[2004-2008] Project Engineer, Laboratory of Bioinformatics and Mathematics of the Genome, Universidad de Chile, Santiago, Chile.
  \item[2008-2012] Fulbright fellow.
  \item[2014] Doctor of Philosophy, Scripps Institution of Oceanography, University of California, San Diego 
\end{vita}


\begin{publications}

\item Martin LJ, Adams R, Bateman A, Bik HM, Haws J, Hird SM, Hughes D, Kembel SW, Kinney K, Kolokotronis SO, Levy G, Mclain C, Meadow JF, Medina RF, Mhuireach G, Moreau CS, Munshi-South J, Nichols, LM, Palmer C, Popova L, Schal C, Siegel J, Taubel M, Trautwein M, Ugalde JA, Dunn RR. Evolution in the Indoor Biome. \emph{Proc R Soc B}. \emph{Accepted}.

  \item Valenzuela C, Ugalde JA, Mora GC, Alvarez S, Contreras I, Santiviago CA. Draft Genome Sequence of \emph{Salmonella enterica} Serovar \emph{Typhi}.  \emph{Genome Announc} 2(1): e00104-14. 2014.

  \item Malfatti F, Turk V, Tinta T, Mozeti\v{c} P, Manganelli M, Samo TJ, Ugalde JA, Kova\v{c} N, Stefanelli M, Antonioli M, Fonda-Umani S, Del Negro P, Cataletto B, Hozi\'{c} A, Ivo\v{s}evi\'{c} DeNardis N, Mi\v{s}i\'{c} Radi\'{c} T, Radi\'{c} T, Fuks D, Azam F. Microbial mechanisms coupling carbon and phosphorus cycles in phosphorous-limited northern Adriatic Sea. \emph{Sci Total Environ} 470: 1173-1183. 2014.
  
  \item Ugalde JA, Narasingarao P, Kuo P, Podell S, Allen EE. Draft genome sequence of "\emph{Candididatus} Halobonum tyrrellensis" Strain G22, Isolated from the Hypersaline Waters of Lake Tyrrell, Australia.  \emph{Genome Announc} 1(6): e01001-13. 2013.
  
  \item Podell S, Emerson JB, Jones, CM, Ugalde JA, Welch, S, Heidelberg KB, Banfield JF, Allen EE. Seasonal fluctuations in ionic concentrations drive microbial succession in a hypersaline lake community. \emph{ISME J}, advance online publication. doi:10.1038/ismej.2013.221.
  
  \item Kharbush JJ, Ugalde JA, Hogle SL, Allen EE, Aluwihare LI. Composite bacterial hopanoids and their microbial producers across oxygen gradients in the water column of the California Current. \emph{Appl Env Microbiol} 79(23): 7491-7501. 2013.
  
  \item Ugalde JA, Gallard MJ, Belmar C, Mu\~{n}oz P, Ruiz-Tagle N, Ferrada-Fuentes S, Espinoza C, Allen EE, Gallardo VA. Microbial Life in a Fjord: Metagenomic Analysis of a Microbial Mat in Chilean Patagonia. \emph{PLoS One} 8(8):e71952. 2013.
  
  \item Podell S, Ugalde JA, Narasingarao P, Banfield JF, Heidelberg EE. Assembly-driven community genomics of a hypersaline microbial ecosystem. \emph{PLoS One} 8(4):e61692. 2013.
  
   \item Narasingaro P, Podell S, Ugalde JA, Brochier-Armanet C, Emerson JB, Brocks JJ, Heidelberg KB, Banfield JF, Allen EE. De novo metagenomic assembly reveals abundant novel major lineage of Archaea in hypersaline microbial communities. \emph{ISME J} 6:81-93. 2012.
  
  \item Ugalde JA, Podell S, Narasingarao P, Allen EE. Xenorhodopsins, an enigmatic new class of microbial rhodopsins horizontally transferred between Archaea and Bacteria. \emph{Biol Direct} 6:52. 2011.
  
  \item Levic\'{a}n G, Ugalde JA, Ehrenfeld N, Maass A, Parada P. Comparative genomic analysis of carbon and nitrogen assimilation mechanisms in three indigenous bioleaching bacteria: predictions and validations. \emph{BMC Genomics} 9(1):581. 2008.
  
  \item Chang BSW, Ugalde JA, Matz MV. Applications of ancestral protein reconstruction in understanding protein function: GFP-like proteins. \emph{Meth Enyzmol} 395:652-670. 2005.
  
  \item Matz MV, Labas YA, Ugalde J. Evolution of function and color in GFP-like proteins. \emph{Method of Biochemical Analysis, Green Fluorescent Protein}. Chalfie M, Kain SR, Eds. John Wiley \& Sons. 2005.
  
  \item Ugalde JA, Chang BSW, Matz MV. Evolution of Coral Pigments Recreated. \emph{Science} 305(5689): 1433. 2004.
  
  \item Shagin DA, Barsova EV, Yanushevich YG, Fradkov AF, Lukyanov KA, Labas YA, Semenova TN, Ugalde JA, Meyers A, Nunez JM, Widder EA, Lukyanov SA, Matz MV. GFP-like proteins as ubiquitous metazoan superfamily: evolution of functional features and structural complexity. \emph{Mol Biol Evol} 21(5): 841-850. 2004.
  
\end{publications}
\end{vitapage}


%% ABSTRACT
%
%  Doctoral dissertation abstracts should not exceed 350 words. 
%   The abstract may continue to a second page if necessary.
%
\begin{abstract}

The use of metagenomic approximations to study natural microbial communities has allowed us to understand the phylogenetic and functional composition of these communities. Gene fragments, derived from the sequence reads, can be phylogenetically classified by binning methods, and compared against reference databases for functional assignments. However, these classifications are limited by the availability of reference genomes in the databases. One way to overcome such limitations, is through the use of assembly-based metagenomics, where the \textit{de novo} sequence assembly, which does not rely on external reference sequences, can allow us to identify novel organisms that are present in the community. 

The work presented here, shows the results of the assembly-based metagenomic characterization of a hypersaline microbial community, from Lake Tyrrell, Australia. The main objective was the reconstruction of the most abundant members of this microbial community using the sequence information, and generate a set of habitat-specific genomes that can be used for future studies.

The assembly and characterization of this metagenomic dataset allowed the discovery of a novel archaeal Class, the \textit{Nanohaloarchaea} (Chapter 2), an ubiquitous lineage late found to be present in other hypersaline environments.
%Change this to Phylum?
%DOUG:p.xxi. “discovery of a novel archaeal Class, the Nanohaloarchaea”.  As you indicate later on this is now understood to be a new division.

The work on the \textit{Nanohaloarchaea}, led to the discovery a novel type of rhodopsin protein, Xenorhodopsins (Chapter 3), that is present on the genomes of members of this group. Phylogenetic analysis indicates that this new rhodopsin has been horizontally transferred between Bacteria and Archaea.

The complete assembly analysis of the metagenome dataset, allowed the description of the most abundante members present in this microbial community (Chapter 4), allowing estimations of relative abundance, phylogenetic and functional diversity.

With the availability of habitat-specific genomes, it is possible to study not only the phylogenetic and functional diversity, but also the fine-scale genetic diversity of the members of the community. Deep sampling of four samples from the Lake Tyrrell microbial community, using high-throughput sequencing, and the availability of habitat-specific genomes allowed the characterization of this genetic diversity (Chapter 5). This information was used to compare the genetic diversity between populations, and identify signatures of environmental adaptation at the sequence level.


\end{abstract}


\end{frontmatter}




















