{
 "metadata": {
  "name": ""
 },
 "nbformat": 3,
 "nbformat_minor": 0,
 "worksheets": [
  {
   "cells": [
    {
     "cell_type": "markdown",
     "metadata": {},
     "source": [
      "This notebook has the code used to process the SNP files generated by the AMOS processes pipeline. To generate these files, the assemblies of the J07 and A07 populations (generated using the Celera Assembler), were proccessed using the analyzeSNPs script from AMOS.\n",
      "\n",
      "The parameteres used for analyzeSNPs, were:\n",
      "- Minimum depth of 4\n",
      "- At least two different bases of consensus\n",
      "- Quality score of 20 or more\n",
      "\n",
      "For those assemblies that had 454 reads, those reads were not considered in the analysis. Only SNPs supported by Sanger reads were considered.\n"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Import the modules needed\n",
      "from collections import defaultdict\n",
      "import numpy as np\n",
      "%matplotlib inline\n",
      "import matplotlib.pyplot as plt\n",
      "import matplotlib.mlab as mlab\n",
      "import seaborn as sns\n",
      "from collections import defaultdict\n",
      "import re\n",
      "from Bio import SeqIO"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 1
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Read the list of scaffolds and store in a dictionary\n",
      "#The structure is: key-> Assembly_ID\n",
      "#                  values -> [JGI_ID, length]\n",
      "def read_scaffold_list(input):\n",
      "    \n",
      "    output_dic = defaultdict(list)\n",
      "    \n",
      "    for line in open(input, 'r'):\n",
      "        if line.strip():\n",
      "            line = line.rstrip()\n",
      "            info = line.split(\"\\t\")\n",
      "            output_dic[info[0]] = [info[2],info[1]]\n",
      "    \n",
      "    return output_dic"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 2
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Read the contig-scaffold mapping, store in dictionary\n",
      "#The structure is: key->contig_id\n",
      "#Values:           values->[scaf_id,start,end,orientation]\n",
      "\n",
      "def read_ctg_scaf_map(input):\n",
      "    output_dic = defaultdict(list)\n",
      "    \n",
      "    for line in open(input, 'r'):\n",
      "        if line.strip():\n",
      "            line = line.rstrip()\n",
      "            info = line.split(\"\\t\")\n",
      "            output_dic[info[0]] = [info[1],info[2],info[3],info[4]]\n",
      "    \n",
      "    return output_dic"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 3
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Read the AMOS SNP file, change info according to the scaffolds with the JGI ID and store\n",
      "#the information ready for the VCF file\n",
      "#Dictionary: key->scaffold\n",
      "#            values: [position,id,etc..... (rest of columns VCF)\n",
      "\n",
      "def read_amos_snp(scaf_list,mapping,input_snp,genome_file):\n",
      "    logfile = open(\"logfile.txt\", 'w')\n",
      "    \n",
      "    #Dictionary to get the complementary nucleotide\n",
      "    complement_dict = {\"A\":\"T\",\"T\":\"A\",\"C\":\"G\",\"G\":\"C\"}\n",
      "    \n",
      "    \n",
      "    fasta_sequences = SeqIO.parse(open(genome_file), \"fasta\")\n",
      "    genome_dictionary = defaultdict(str)\n",
      "    \n",
      "    for fasta in fasta_sequences:\n",
      "        name, sequence = fasta.id, fasta.seq.tostring()\n",
      "        genome_dictionary[name] = sequence\n",
      "          \n",
      "    \n",
      "    output_vcf_results = defaultdict(lambda: defaultdict(list))\n",
      "    snp_count = defaultdict(int)\n",
      "    \n",
      "    for line in open(input_snp, 'r'):\n",
      "        \n",
      "        if line.startswith(\"AsmblID\"):\n",
      "            continue\n",
      "        if line.strip():\n",
      "            \n",
      "            line = line.rstrip()\n",
      "            info = line.split(\"\\t\")\n",
      "            \n",
      "            ctg_id = info[0]\n",
      "            ungapped_position = info[2]\n",
      "            consensus_base = info[3]\n",
      "            depth_coverage = info[4]\n",
      "            \n",
      "            bases_info = info[6:]\n",
      "            \n",
      "            #Skip gaps, because we are looking for SNPs, and insertion/deletions complicate the creation\n",
      "            #of the VCF file\n",
      "            if consensus_base == \"-\":\n",
      "                continue\n",
      "            \n",
      "            #Skip if coverage is less than four:\n",
      "            if not int(depth_coverage) > 3:\n",
      "                continue\n",
      "            \n",
      "            #Skip those contig with no mapping info\n",
      "            if not ctg_id in mapping:\n",
      "                continue\n",
      "            \n",
      "            \n",
      "            #Only look at the scaffolds that are part of the assembly\n",
      "            scf_id,start,end,orientation = mapping[ctg_id]\n",
      "            if not scf_id in scaf_list:\n",
      "                continue\n",
      "                \n",
      "            jgi_scf_id = scaf_list[scf_id][0]\n",
      "            \n",
      "            #Calculate the SNP position in the scaffold\n",
      "            snp_position_scf = 0\n",
      "            \n",
      "            if orientation == 'f':\n",
      "                snp_position_scf = int(start) + int(ungapped_position)\n",
      "            \n",
      "            if orientation == 'r':\n",
      "                snp_position_scf = int(end) - int(ungapped_position) + 1\n",
      "            \n",
      "            \n",
      "            #Get the depth of each base\n",
      "            bases_dict = defaultdict(int)\n",
      "            adjusted_depth = 0 \n",
      "            \n",
      "            \n",
      "            for base in bases_info:\n",
      "                number_search = re.match('(\\D+)\\((\\d+)\\)',base)\n",
      "                \n",
      "                if int(number_search.group(2)) > 2:                    \n",
      "                    base = str()\n",
      "                    \n",
      "                    if orientation == 'r':\n",
      "                        \n",
      "                        if number_search.group(1) in complement_dict:    \n",
      "                            base = complement_dict[number_search.group(1)]\n",
      "                        else:\n",
      "                            base = number_search.group(1)\n",
      "                    \n",
      "                    if orientation == 'f':\n",
      "                        base = number_search.group(1)\n",
      "                    \n",
      "                    \n",
      "                    bases_dict[base] = number_search.group(2)\n",
      "                    adjusted_depth += int(number_search.group(2))\n",
      "                    \n",
      "            gap_counter = 0\n",
      "            for base in bases_dict:\n",
      "                if base == \"-\":\n",
      "                    gap_counter += 1\n",
      "            \n",
      "            if not gap_counter == 0:\n",
      "                continue\n",
      "                \n",
      "                    \n",
      "            \n",
      "            \n",
      "            #Because some of the assemblies are based in 454 data, I need to check that the consensus\n",
      "            #is indeed the deepest base. If not, I just move to the next one. \n",
      "            #This is stringent, but the idea is to look at SNPs that are supported by the sanger reads\n",
      "            #The details of the population structure should come out from the Illumina reads\n",
      "         \n",
      "            if not len(bases_dict.keys()) > 1:\n",
      "                continue\n",
      "            \n",
      "    \n",
      "\n",
      "            #Values for the final dictionary\n",
      "            #Chrom\n",
      "            #Pos\n",
      "            #ID, always .\n",
      "            #REF\n",
      "            #ALT If multiple, separated by , Replace \"-\" with \".\"\n",
      "            #Qual, always 20\n",
      "            #Filter, always PASS\n",
      "            #INFO: #NS=1,DP=depth,AF=allele frequency\n",
      "            \n",
      "            #For some reason, AMOS gives position that are not the ones found in the consensus\n",
      "            #Here I will check against the fasta file for the sequence, to see if the position correspond to the consensus\n",
      "            #or the alternates\n",
      "            \n",
      "            \n",
      "                        \n",
      "            #Create the alt and allele frequency values\n",
      "            #genome_reference_nuc = genome_sequence[jgi_scf_id].seq[int(snp_position_scf) + 1]\n",
      "            reference_position = int(snp_position_scf) - 1\n",
      "            \n",
      "            try:\n",
      "                genome_reference_nuc = genome_dictionary[jgi_scf_id][reference_position]\n",
      "            except IndexError:\n",
      "                logfile.write(\"Error: %s\\t%d\\n\" %(jgi_scf_id,reference_position))\n",
      "                continue\n",
      "                \n",
      "            new_reference_position = \".\"\n",
      "            alternative_bases = []\n",
      "            allele_frequency = []\n",
      "            \n",
      "            \n",
      "            for entry in bases_dict.keys():\n",
      "                \n",
      "                if entry == genome_reference_nuc:\n",
      "                    new_reference_position = entry\n",
      "                    \n",
      "                else:\n",
      "                    alternative_bases.append(entry)\n",
      "                    freq = int(bases_dict[entry]) / float(adjusted_depth)\n",
      "                    allele_frequency.append(freq)\n",
      "            \n",
      "            #Now save everything into an entry\n",
      "            \n",
      "            vcf_alt = \",\".join(alternative_bases)\n",
      "            info_entry = \"NS=1,\" + \"DP=\" + str(adjusted_depth) + \",AF=\" + \",\".join(map(str, allele_frequency))\n",
      "            \n",
      "            #Modify consensus base to \".\"\n",
      "            #if consensus_base == \"-\":\n",
      "            #    consensus_base = \".\"\n",
      "            \n",
      "            vcf_entry = [\".\", new_reference_position, vcf_alt, \"20\", \"PASS\", info_entry]\n",
      "            \n",
      "            snp_count[scf_id] += 1\n",
      "            \n",
      "            output_vcf_results[jgi_scf_id][snp_position_scf] = vcf_entry\n",
      "            \n",
      "            logfile.write(orientation + \"\\t\" + \",\".join(bases_dict.keys()) + \"\\t\" + genome_reference_nuc + \",\" + new_reference_position +\"\\t\" + jgi_scf_id +\"\\t\" + str(snp_position_scf) + \"\\t\" + \",\".join(vcf_entry) + \"\\n\")\n",
      "                                \n",
      "    logfile.close()\n",
      "    return output_vcf_results,snp_count\n",
      "            \n",
      "            \n",
      "                        \n",
      "            \n",
      "                            \n",
      "\n",
      "\n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 4
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "def write_vcf(snp_info,file):\n",
      "    #This function will write a vcf file based on the generated snp dictionary\n",
      "    import datetime\n",
      "    today = datetime.date.today()\n",
      "    \n",
      "    outfile = open(file, 'w')\n",
      "    \n",
      "    outfile.write(\"##fileformat=VCFv4.1\\n\")\n",
      "    outfile.write(\"##filedate=%s%s%s\\n\" % (today.year,today.month,today.day))\n",
      "    outfile.write(\"##source=AMOS_file_JU\\n\")\n",
      "    outfile.write(\"##INFO=<ID=NS,Number=1,Type=Integer,Description=\\\"Number of samples with data\\\">\\n\")\n",
      "    outfile.write(\"##INFO=<ID=DP,Number=1,Type=Integer,Description=\\\"Total Depth\\\">\\n\")\n",
      "    outfile.write(\"##INFO=<ID=AF,Number=A,Type=Float,Description=\\\"Allele Frequency\\\">\\n\")\n",
      "    outfile.write(\"#CHROM\\tPOS\\tID\\tREF\\tALT\\tQUAL\\tFILTER\\tINFO\\n\")\n",
      "    \n",
      "    for scaffold in sorted(snp_info):\n",
      "        for position in sorted(snp_info[scaffold]):\n",
      "        \n",
      "            outfile.write(scaffold + \"\\t\" + str(position) + \"\\t\" + \"\\t\".join(map(str,snp_info[scaffold][position])) + \"\\n\")\n",
      "        \n",
      "    \n",
      "    outfile.close()\n",
      "    \n",
      "    "
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 5
    },
    {
     "cell_type": "heading",
     "level": 3,
     "metadata": {},
     "source": [
      "Store the results for global comparison among the genomes"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "genome_results = defaultdict(list)"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 6
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07ABHN4"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07ABHN4\n",
      "genome_folder = \"J07ABHN4\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HN4/2512875007.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/454_Trimmed.HN4_v3_031612_readnames_Q20C20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07ABHN4.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/HN4_v3_031612.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 7
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 8
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07HN4\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    "
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07HN4v2_scf7180000001347.1\t547036\t961\n",
        "J07HN4v2_scf7180000001348.2\t2341623\t7958\n"
       ]
      }
     ],
     "prompt_number": 9
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07HN4_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 10
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HN4 \\\n",
      "$output_vcf_file > $genome_folder/J07HN4_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t1064\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 11
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07ABHN6"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07ABHN6\n",
      "genome_folder = \"J07ABHN6\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HN6/2512875008.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/454Trimmed_HN6_031912_readnames_Q20C20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07ABHN6.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/HN6_031912.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 12
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07HN6\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07HN6_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n",
      "\n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07HN6v2_scf7180000001851.4\t424200\t2746\n",
        "J07HN6v2_scf7180000001850.3\t185112\t1476\n",
        "J07HN6v2_scf7180000001853.6\t896196\t5442\n",
        "J07HN6v2_scf7180000001852.5\t873166\t6434\n",
        "J07HN6v2_scf7180000001848.1\t61036\t602\n",
        "J07HN6v2_scf7180000001849.2\t89290\t553\n"
       ]
      }
     ],
     "prompt_number": 13
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HN6 \\\n",
      "$output_vcf_file > $genome_folder/J07HN6_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t1969\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 14
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07ABHX64"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07ABHX64\n",
      "genome_folder = \"J07ABHX64\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HX64/2502082092.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = \"./AC_sang_080509mer15_C20Q20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07ANHX64.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = \"./AC_sang_080509mer15.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 15
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07ABHX64\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07ABHX64_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07ABHX6_J07ABscf098875\t2982938\t5466\n"
       ]
      }
     ],
     "prompt_number": 16
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HX64 \\\n",
      "$output_vcf_file > $genome_folder/J07HX64_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t997\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 17
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07ABHX67"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07ABHX67\n",
      "genome_folder = \"J07ABHX67\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HB67/2506783034.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = \"./AC_sang_080509mer15_C20Q20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07ABHX67.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = \"./AC_sang_080509mer15.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 18
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07ABHX67\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07ABHX67_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07ABHX67v2__Contig_2\t254249\t145\n",
        "J07ABHX67v2__Contig_1\t110024\t62\n",
        "J07ABHX67v2__Contig_3\t2285274\t2851\n"
       ]
      }
     ],
     "prompt_number": 19
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HX67 \\\n",
      "$output_vcf_file > $genome_folder/J07HX67_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t1414\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 20
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07HQX"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07HQX\n",
      "genome_folder = \"J07HQX\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HQX/2512875009.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/454Trimmed_HQX_031812_readnames_Q20C20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07HQX.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/HQX_031812.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 21
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07HQX\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07HQX_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07HQXv2_scf7180000001541.2\t1476021\t131\n",
        "J07HQXv2_scf7180000001540.1\t1543888\t28\n"
       ]
      }
     ],
     "prompt_number": 22
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HQX \\\n",
      "$output_vcf_file > $genome_folder/J07HQX_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t15\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 23
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07HWQ1"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07HWQ1\n",
      "genome_folder = \"J07HWQ1\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HQW1/2512875005.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/AC_0.8sang_080309mer15.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07HWQ1.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/AC_0.8sang_080309mer15.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 24
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07HWQ1\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07HWQ1_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07HWQ1_J07B_scf56329a.1\t3475501\t24677\n"
       ]
      }
     ],
     "prompt_number": 25
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HWQ1 \\\n",
      "$output_vcf_file > $genome_folder/J07HWQ1_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t2339\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 26
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07HWQ2"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07HWQ1\n",
      "genome_folder = \"J07HWQ2\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HQW2/2512875006.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/HQW2_08err_032412a_Q20C20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07HWQ2.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/HQW2_08err_032412a.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 27
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07HWQ2\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07HWQ2_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07HQW2_scf7180000002443.1\t3594539\t13863\n"
       ]
      }
     ],
     "prompt_number": 28
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HWQ2 \\\n",
      "$output_vcf_file > $genome_folder/J07HWQ2_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t1233\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 29
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07HR59"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07HR59\n",
      "genome_folder = \"J07HR59\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HR59/2512875011.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/454Trimmed_HR_032012_readnames_Q20C20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07HR59.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/HR_032012.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 30
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07HR59\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07HR59_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07HR59_scf7180000001381.6\t184023\t7\n",
        "J07HR59_scf7180000001380.5\t49857\t29\n",
        "J07HR59_scf7180000001382.7\t1672266\t255\n",
        "J07HR59_scf7180000001326.1\t72446\t1\n"
       ]
      }
     ],
     "prompt_number": 31
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HR59 \\\n",
      "$output_vcf_file > $genome_folder/J07HR59_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t64\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 32
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07HX5"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07HX5\n",
      "genome_folder = \"J07HX5\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07HX5/2512875010.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/454_Trimmed.HX64_HX5_031512_Q20C20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07HX5.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/HX64_HX5_031512.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 33
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07HX5\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07HX5_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07HX5_scf7180000022092.1\t2040945\t2046\n"
       ]
      }
     ],
     "prompt_number": 34
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07HX5 \\\n",
      "$output_vcf_file > $genome_folder/J07HX5_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t352\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 35
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07NFR43"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07NFR43\n",
      "genome_folder = \"J07NFR43\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07AB43/2502422326.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/454Trimmed_J07NFR43_readnames_Q20C20M2.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07NFR43.scaffold_list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/J07NFR43.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 36
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07NFR43\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07NFR43_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07NFR4_J07NFR43scf30742\t52428\t885\n",
        "J07NFR4_J07NFR43scf30739\t112863\t2281\n",
        "J07NFR4_J07NFR43scf30734\t65032\t857\n",
        "J07NFR4_J07NFR43scf30737\t32088\t201\n",
        "J07NFR4_J07NFR43scf30744\t798418\t7669\n",
        "J07NFR4_J07NFR43scf30726\t111825\t701\n",
        "J07NFR4_J07NFR43scf30724\t54503\t224\n"
       ]
      }
     ],
     "prompt_number": 37
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07AB43 \\\n",
      "$output_vcf_file > $genome_folder/J07AB43_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t2174\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 38
    },
    {
     "cell_type": "heading",
     "level": 2,
     "metadata": {},
     "source": [
      "J07NFR56"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Files for J07NFR56\n",
      "genome_folder = \"J07NFR56\"\n",
      "\n",
      "#Fasta file of the genome (nucleotide)\n",
      "fasta_genome = \"../jgi_genomes/J07AB56/2502422327.fna\"\n",
      "\n",
      "#SNP file, generated by amos\n",
      "amos_snp_file = genome_folder + \"/454Trimmed_J07NFR56_Q20C20.snps\"\n",
      "\n",
      "#File with the list of scaffolds. The format is:\n",
      "#ID_Assembly Length ID_JGI\n",
      "scaffold_list = genome_folder + \"/J07NFR56.scaffold.list\"\n",
      "\n",
      "#Contig to Scaffold Mapping. The SNP file that AMOS generates, has the coordinates by contig.\n",
      "#A second file is needed to map this coordinates to the scaffolds. Format:\n",
      "#Contig Scaffold Start End Orientation\n",
      "ctg_scaf_map = genome_folder + \"/J07NFR56.posmap.ctgscf\""
     ],
     "language": "python",
     "metadata": {},
     "outputs": [],
     "prompt_number": 39
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Generate the list of the snps, and a general count of SNPs found on each scaffold\n",
      "genome_scaffolds = read_scaffold_list(scaffold_list)\n",
      "mapping_info = read_ctg_scaf_map(ctg_scaf_map)\n",
      "output_vcf_list,snp_count = read_amos_snp(genome_scaffolds, mapping_info, amos_snp_file, fasta_genome)\n",
      "\n",
      "#Print some basic stats on number of snps and length\n",
      "for scaf in snp_count:\n",
      "    print genome_scaffolds[scaf][0] + \"\\t\" + str(genome_scaffolds[scaf][1]) + \"\\t\" + str(snp_count[scaf])\n",
      "    \n",
      "    genome_results[\"J07NFR56\"] = [genome_scaffolds[scaf][0], str(genome_scaffolds[scaf][1]), str(snp_count[scaf])]\n",
      "    \n",
      "    #Store the results for comparisons with the other genomes\n",
      "    \n",
      "#Write the VCF file\n",
      "output_vcf_file = genome_folder + \"/J07NFR56_Assembly_snps.vcf\"\n",
      "write_vcf(output_vcf_list, output_vcf_file)\n",
      "    \n"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "J07NFR5_J07NFR56scf39101\t959093\t6809\n",
        "J07NFR5_J07NFR56scf39097\t60285\t196\n",
        "J07NFR5_J07NFR56scf39072\t196424\t1527\n"
       ]
      }
     ],
     "prompt_number": 40
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "#Run snpEFF\n",
      "!/Library/Internet\\ Plug-Ins/JavaAppletPlugin.plugin/Contents/Home/bin/java -jar ~/BioApps/snpEff/snpEff.jar eff \\\n",
      "-no-downstream -no-upstream -no-utr -ud 0 -o vcf -c snpEff.config J07AB56 \\\n",
      "$output_vcf_file > $genome_folder/J07AB56_Assembly_snpEFF.vcf\n",
      "\n",
      "!mv snpEff_summary.html $genome_folder/\n",
      "!mv snpEff_genes.txt $genome_folder/"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "\r\n",
        "WARNINGS: Some warning were detected\r\n",
        "Warning type\tNumber of warnings\r\n",
        "WARNING_TRANSCRIPT_NO_START_CODON\t3875\r\n",
        "\r\n",
        "\r\n"
       ]
      }
     ],
     "prompt_number": 41
    },
    {
     "cell_type": "heading",
     "level": 1,
     "metadata": {},
     "source": [
      "Data Summary"
     ]
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [
      "print genome_results"
     ],
     "language": "python",
     "metadata": {},
     "outputs": [
      {
       "output_type": "stream",
       "stream": "stdout",
       "text": [
        "defaultdict(<type 'list'>, {'J07NFR56': ['J07NFR5_J07NFR56scf39072', '196424', '1527'], 'J07HX5': ['J07HX5_scf7180000022092.1', '2040945', '2046'], 'J07HN6': ['J07HN6v2_scf7180000001849.2', '89290', '553'], 'J07HN4': ['J07HN4v2_scf7180000001348.2', '2341623', '7958'], 'J07HR59': ['J07HR59_scf7180000001326.1', '72446', '1'], 'J07HWQ2': ['J07HQW2_scf7180000002443.1', '3594539', '13863'], 'J07HQX': ['J07HQXv2_scf7180000001540.1', '1543888', '28'], 'J07HWQ1': ['J07HWQ1_J07B_scf56329a.1', '3475501', '24677'], 'J07ABHX67': ['J07ABHX67v2__Contig_3', '2285274', '2851'], 'J07NFR43': ['J07NFR4_J07NFR43scf30724', '54503', '224'], 'J07ABHX64': ['J07ABHX6_J07ABscf098875', '2982938', '5466']})\n"
       ]
      }
     ],
     "prompt_number": 42
    },
    {
     "cell_type": "code",
     "collapsed": false,
     "input": [],
     "language": "python",
     "metadata": {},
     "outputs": []
    }
   ],
   "metadata": {}
  }
 ]
}